<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>EMERGENT</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.79.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css rel=stylesheet><link rel=stylesheet href=/css/custom.css><link href=https://read-lab-confederation.github.io/index.xml rel=alternate type=application/rss+xml title=EMERGENT><link href=https://read-lab-confederation.github.io/index.xml rel=feed type=application/rss+xml title=EMERGENT><meta property="og:title" content="EMERGENT"><meta property="og:description" content="EMory bactERial GEnomics aNd meTagenomics"><meta property="og:type" content="website"><meta property="og:url" content="https://read-lab-confederation.github.io/"><meta property="og:updated_time" content="2023-05-18T00:00:00+00:00"><meta itemprop=name content="EMERGENT"><meta itemprop=description content="EMory bactERial GEnomics aNd meTagenomics"><meta name=twitter:card content="summary"><meta name=twitter:title content="EMERGENT"><meta name=twitter:description content="EMory bactERial GEnomics aNd meTagenomics"><script type=application/javascript>var doNotTrack=false;if(!doNotTrack){window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;ga('create','UA-148959477-1','auto');ga('send','pageview');}</script><script async src=https://www.google-analytics.com/analytics.js></script><link href=/css/share-button.css rel=stylesheet></head><body class="ma0 avenir bg-near-white"><header class="cover bg-top" style=background-image:url(https://read-lab-confederation.github.io/images/USA500_circle.png)><div class=bg-black-60><nav class="pv3 ph3 ph4-ns tc tl-l" role=navigation><div class="flex-l justify-between items-center center"><a href=https://read-lab-confederation.github.io class="f3 fw2 hover-white no-underline white-90 dib">EMERGENT</a><div class="flex-l items-center"><ul class="pl0 mr3"><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/blog/ title="Blog page">Blog</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/group/ title="Group page">Group</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/research/ title="Research page">Research</a></li><li class="list f5 f4-ns fw4 dib pr1 pr3-ns"><a href=https://twitter.com/tdread_emory target=_blank class="link-transition twitter link dib z-999 pt0-l mr1" title="Twitter link" rel=noopener aria-label="follow on Twitter——Opens in a new window"><svg height="24" style="enable-background:new 0 0 67 67" viewBox="0 0 67 67" width="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M37.167 22.283c-2.619.953-4.274 3.411-4.086 6.101l.063 1.038-1.048-.127c-3.813-.487-7.145-2.139-9.974-4.915l-1.383-1.377-.356 1.017c-.754 2.267-.272 4.661 1.299 6.271.838.89.649 1.017-.796.487-.503-.169-.943-.296-.985-.233-.146.149.356 2.076.754 2.839.545 1.06 1.655 2.097 2.871 2.712l1.027.487-1.215.021c-1.173.0-1.215.021-1.089.467.419 1.377 2.074 2.839 3.918 3.475l1.299.444-1.131.678c-1.676.976-3.646 1.526-5.616 1.568C19.775 43.256 19 43.341 19 43.405c0 .211 2.557 1.397 4.044 1.864 4.463 1.377 9.765.783 13.746-1.568 2.829-1.673 5.657-5 6.978-8.221.713-1.716 1.425-4.851 1.425-6.354.0-.975.063-1.102 1.236-2.267.692-.678 1.341-1.419 1.467-1.631.21-.403.188-.403-.88-.043-1.781.636-2.033.551-1.152-.402.649-.678 1.425-1.907 1.425-2.267.0-.063-.314.042-.671.233-.377.212-1.215.53-1.844.72l-1.131.361-1.027-.7c-.566-.381-1.361-.805-1.781-.932C39.766 21.902 38.131 21.944 37.167 22.283zM33 64C16.432 64 3 50.569 3 34S16.432 4 33 4s30 13.431 30 30S49.568 64 33 64z" style="fill-rule:evenodd;clip-rule:evenodd;fill:"/></svg><span class=new-window><svg height="8" style="enable-background:new 0 0 1000 1000" viewBox="0 0 1e3 1e3" width="8" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M598 128h298v298h-86V274L392 692l-60-60 418-418H598v-86zM810 810V512h86v298c0 46-40 86-86 86H214c-48 0-86-40-86-86V214c0-46 38-86 86-86h298v86H214v596h596z" style="fill-rule:evenodd;clip-rule:evenodd;fill:"/></svg></span></a></li></ul></div></div></nav><div class="tc pv2-l pb2 pv6-l ph2 ph4-ns"><h1 class="f2-l f5 f-subheadline-l fw2 white-90 mb0 lh-title dn dib-ns">EMERGENT</h1><h2 class="fw1 f7 f3-l white-80 measure-wide-l center mt3">EMory bactERial GEnomics aNd meTagenomics</h2></div></div></header><main class=pb7 role=main><article class="cf ph3 ph5-l pv3 pv4-l f4 tc-l center measure-wide lh-copy mid-gray"></article><div class="pa3 pa4-ns w-100 w-70-ns center"><h1 class="f1-ns f5 flex-none">Recent Posts</h1><section class="w-100 mw8"><div class="relative w-100 mb4"><article class="bb b--black-10"><div class="db pv4 ph3 ph0-l no-underline dark-gray"><div class="flex flex-column flex-row-ns"><div class="pr3-ns mb4 mb0-ns w-100 w-40-ns"><a href=/blog/posts/parsing_ncbi_antibiograms/ class="db grow"><img src=https://read-lab-confederation.github.io/images/blog202210_Image1.jpg class=img alt="image from How I freed NCBI antibiogram data from XML"></a></div><div class="blah w-100 w-60-ns pl3-ns"><h1 class="f3-ns f5 fw1 athelas mt0 lh-title"><a href=/blog/posts/parsing_ncbi_antibiograms/ class="color-inherit dim link">How I freed NCBI antibiogram data from XML</a></h1><span class="f6-ns f7 db"><div class=author-info><p class=author-name>Brooke Talbot
& Vishnu Raghuram
                </p></div></span><span class="f6-ns f7 db">May 18, 2023</span><br><div class="f7 f5-l lh-copy nested-copy-line-height nested-links">How I freed NCBI antibiogram data from XML TL;DR How I made a tidy data frame of >24,000 bacterial BioSample IDs with antibiograms and linked genomic sequence data using a customized R script. Code and data available on the github site. Antibiograms Antibiotic resistant bacteria continue to become more common worldwide. Diagnosis of antibiotic resistance is a frontline defense for fighting back against these dangerous pathogens. Antibiograms are phenotypic tests used routinely in clinical settings to ascertain the profile of antibiotics that will effectively treat the infection.</div><a href=/blog/posts/parsing_ncbi_antibiograms/ class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a></div></div></div></article></div><div class="relative w-100 mb4"><article class="bb b--black-10"><div class="db pv4 ph3 ph0-l no-underline dark-gray"><div class="flex flex-column flex-row-ns"><div class="pr3-ns mb4 mb0-ns w-100 w-40-ns"><a href=/blog/posts/reference_staph_aureus_blog_vr_edits/ class="db grow"><img src=https://read-lab-confederation.github.io/images/blog202210_Image1.jpg class=img alt="image from Meet the Staph: A menagerie of common S. aureus reference strains"></a></div><div class="blah w-100 w-60-ns pl3-ns"><h1 class="f3-ns f5 fw1 athelas mt0 lh-title"><a href=/blog/posts/reference_staph_aureus_blog_vr_edits/ class="color-inherit dim link">Meet the Staph: A menagerie of common <em>S. aureus</em> reference strains</a></h1><span class="f6-ns f7 db"><div class=author-info><p class=author-name>Brooke Talbot
& Vishnu Raghuram
                </p></div></span><span class="f6-ns f7 db">October 31, 2022</span><br><div class="f7 f5-l lh-copy nested-copy-line-height nested-links">When we want to answer a specific phenotype question about bacteria, we are often left to ask which strain is the best choice? It’s important to have a variety of strain backgrounds for your laboratory experiments because different genetic backgrounds will help to answer different questions. This extends to all manner of living laboratory organisms from C. elegans to larger vertebrates. In the context of human health research, these higher order model organisms have been bred and manipulated to optimize their specific relevance to modeling a human health concern.</div><a href=/blog/posts/reference_staph_aureus_blog_vr_edits/ class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a></div></div></div></article></div><div class="relative w-100 mb4"><article class="bb b--black-10"><div class="db pv4 ph3 ph0-l no-underline dark-gray"><div class="flex flex-column flex-row-ns"><div class="pr3-ns mb4 mb0-ns w-100 w-40-ns"><a href=/blog/posts/derep_blog/ class="db grow"><img src=https://read-lab-confederation.github.io/images/Vishnu_Fig2.png class=img alt="image from Help, I have too many genome sequences!"></a></div><div class="blah w-100 w-60-ns pl3-ns"><h1 class="f3-ns f5 fw1 athelas mt0 lh-title"><a href=/blog/posts/derep_blog/ class="color-inherit dim link">Help, I have too many genome sequences!</a></h1><span class="f6-ns f7 db"><div class=author-info><p class=author-name>Vishnu Raghuram</p></div></span><span class="f6-ns f7 db">January 21, 2022</span><br><div class="f7 f5-l lh-copy nested-copy-line-height nested-links">Said no one, ever. Right? (tl;dr at bottom) In a recent publication(1), we analysed the quorum sensing operon of > 40,000 Staphylococcus aureus genomes to identify recurring patterns of genetic variation. To infer transmission of specific mutations, we were tasked with constructing phylogenies for multiple clonal lineages, each comprising thousands of isolates(1).
Whole genome maximum likelihood phylogeny construction is a both time and computation-intensive process, especially for large numbers of tips.</div><a href=/blog/posts/derep_blog/ class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a></div></div></div></article></div><div class="relative w-100 mb4"><article class="bb b--black-10"><div class="db pv4 ph3 ph0-l no-underline dark-gray"><div class="flex flex-column flex-row-ns"><div class="pr3-ns mb4 mb0-ns w-100 w-40-ns"><a href=/blog/posts/trawling_public_data/ class="db grow"><img src=https://read-lab-confederation.github.io/images/bacterial_genomes.png class=img alt="image from The joys and perils of trawling through public bacterial genome data"></a></div><div class="blah w-100 w-60-ns pl3-ns"><h1 class="f3-ns f5 fw1 athelas mt0 lh-title"><a href=/blog/posts/trawling_public_data/ class="color-inherit dim link">The joys and perils of trawling through public bacterial genome data</a></h1><span class="f6-ns f7 db"><div class=author-info><p class=author-name>Tim Read</p></div></span><span class="f6-ns f7 db">December 7, 2021</span><br><div class="f7 f5-l lh-copy nested-copy-line-height nested-links">For many bacterial species there are now thousands of genomes in public databases such as the NCBI Short Read Archive (SRA). These huge datasets offer the possibility of large-scale mining to reveal species-wide patterns of genetic variability in a manner analogous to the way that vast biobanks of human genomic data are powering deep GWAS studies and GISAID has opened the door for SARS-CoV2 genomic analysis. Our group developed the Staphopia database of 43,000+ public Staphylococcus aureus genomes in 2017 (Petit and Read 2018) and the Bactopia pipelines for large scale analysis in 2020 (Petit and Read 2020).</div><a href=/blog/posts/trawling_public_data/ class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a></div></div></div></article></div><div class="relative w-100 mb4"><article class="bb b--black-10"><div class="db pv4 ph3 ph0-l no-underline dark-gray"><div class="flex flex-column flex-row-ns"><div class="pr3-ns mb4 mb0-ns w-100 w-40-ns"><a href=/blog/posts/bt-google_grad/ class="db grow"><img src=https://read-lab-confederation.github.io/images/bt-Image_5.jpg class=img alt="image from OK Google, How do I Learn Data Science?"></a></div><div class="blah w-100 w-60-ns pl3-ns"><h1 class="f3-ns f5 fw1 athelas mt0 lh-title"><a href=/blog/posts/bt-google_grad/ class="color-inherit dim link">OK Google, How do I Learn Data Science?</a></h1><span class="f6-ns f7 db"><div class=author-info><p class=author-name>Brooke Talbot</p></div></span><span class="f6-ns f7 db">August 16, 2021</span><br><div class="f7 f5-l lh-copy nested-copy-line-height nested-links">Something that delights me as a data scientist is that concepts and techniques are very readily shared within the community. It’s also astounding to me that each and every day we have access to an infinite body of knowledge which can help supplement formal and informal learning - namely, that we can gain a lot of information very quickly by turning to Professor Google! Investigating how we as scientists search this seemingly endless collection of facts and figures and code blocks, and for what exact purposes, can give some great insights into how we communicate and learn and ultimately improve our reseach.</div><a href=/blog/posts/bt-google_grad/ class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a></div></div></div></article></div></section><section class=w-100><h1 class="f3-ns f5">More</h1><h2 class="f5 fw4 mb4 dib mr3"><a href=/blog/posts/aws-bactopia-intro/ class="link black dim">An introduction to running Bactopia on Amazon Web Services</a></h2><h2 class="f5 fw4 mb4 dib mr3"><a href=/blog/posts/bactopia-aws-and-67000-genomes/ class="link black dim">Using AWS Batch to process 67,000 genomes with Bactopia</a></h2><h2 class="f5 fw4 mb4 dib mr3"><a href=/blog/posts/strain-definition/ class="link black dim">The strain of defining the bacterial strain</a></h2><h2 class="f5 fw4 mb4 dib mr3"><a href=/blog/posts/kyle-michelles_paper/ class="link black dim">Genetics of strain-specific levels of toxin production in *Staphylococcus aureus*</a></h2><a href=/blog/ class="link db f6 pa2 br3 bg-mid-gray white dim w4 tc">All Posts</a></section></div></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://read-lab-confederation.github.io>&copy; 2023 EMERGENT</a><div><a class=resp-sharing-button__link href="https://twitter.com/intent/tweet/?text=Check out EMERGENT, a bacterial genomics and metagenomics blog from @tdread_emory's group!&url=https://read-lab-confederation.github.io" target=_blank rel=noopener aria-label="Share on Twitter"><div class="resp-sharing-button resp-sharing-button--twitter resp-sharing-button--large"><div aria-hidden=true class="resp-sharing-button__icon resp-sharing-button__icon--solid"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M23.44 4.83c-.8.37-1.5.38-2.22.02.93-.56.98-.96 1.32-2.02-.88.52-1.86.9-2.9 1.1-.82-.88-2-1.43-3.3-1.43-2.5.0-4.55 2.04-4.55 4.54.0.36.03.7.1 1.04-3.77-.2-7.12-2-9.36-4.75-.4.67-.6 1.45-.6 2.3.0 1.56.8 2.95 2 3.77-.74-.03-1.44-.23-2.05-.57v.06c0 2.2 1.56 4.03 3.64 4.44-.67.2-1.37.2-2.06.08.58 1.8 2.26 3.12 4.25 3.16C5.78 18.1 3.37 18.74 1 18.46c2 1.3 4.4 2.04 6.97 2.04 8.35.0 12.92-6.92 12.92-12.93.0-.2.0-.4-.02-.6.9-.63 1.96-1.22 2.56-2.14z"/></svg></div>Share on Twitter</div></a></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script><script src=/js/custom.js></script></body></html>