<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>EMERGENT | Research</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.79.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css rel=stylesheet><link rel=stylesheet href=/css/custom.css><meta property="og:title" content="Research"><meta property="og:description" content="Staphylococcus aureus (MRSA) The Gram-positive bacterium S. aureus is one of the leading causes of hospital-acquired infection in the US. Research projects have charactized the genomic epidemiology of clinical isolates and used genome-wide association approaches to understand the genetic basis of antibiotic-resistance, toxin production and phage resistance. We have also developed the “Staphopia” database of whole genome variation in more than 40,000 publically-available sequenced strains.
Chlamydia Bacteria of the Chlamydia genus cause a wide range of infections in humans and animals."><meta property="og:type" content="article"><meta property="og:url" content="https://read-lab-confederation.github.io/research/"><meta property="article:published_time" content="2019-09-09T11:56:20-04:00"><meta property="article:modified_time" content="2019-09-09T11:56:20-04:00"><meta itemprop=name content="Research"><meta itemprop=description content="Staphylococcus aureus (MRSA) The Gram-positive bacterium S. aureus is one of the leading causes of hospital-acquired infection in the US. Research projects have charactized the genomic epidemiology of clinical isolates and used genome-wide association approaches to understand the genetic basis of antibiotic-resistance, toxin production and phage resistance. We have also developed the “Staphopia” database of whole genome variation in more than 40,000 publically-available sequenced strains.
Chlamydia Bacteria of the Chlamydia genus cause a wide range of infections in humans and animals."><meta itemprop=datePublished content="2019-09-09T11:56:20-04:00"><meta itemprop=dateModified content="2019-09-09T11:56:20-04:00"><meta itemprop=wordCount content="256"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="Research"><meta name=twitter:description content="Staphylococcus aureus (MRSA) The Gram-positive bacterium S. aureus is one of the leading causes of hospital-acquired infection in the US. Research projects have charactized the genomic epidemiology of clinical isolates and used genome-wide association approaches to understand the genetic basis of antibiotic-resistance, toxin production and phage resistance. We have also developed the “Staphopia” database of whole genome variation in more than 40,000 publically-available sequenced strains.
Chlamydia Bacteria of the Chlamydia genus cause a wide range of infections in humans and animals."><script type=application/javascript>var doNotTrack=false;if(!doNotTrack){window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;ga('create','UA-148959477-1','auto');ga('send','pageview');}</script><script async src=https://www.google-analytics.com/analytics.js></script><link href=/css/share-button.css rel=stylesheet></head><body class="ma0 avenir bg-near-white"><header><div class=bg-black><nav class="pv3 ph3 ph4-ns tc tl-l" role=navigation><div class="flex-l justify-between items-center center"><a href=https://read-lab-confederation.github.io class="f3 fw2 hover-white no-underline white-90 dib">EMERGENT</a><div class="flex-l items-center"><ul class="pl0 mr3"><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/blog/ title="Blog page">Blog</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/group/ title="Group page">Group</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/research/ title="Research page">Research</a></li><li class="list f5 f4-ns fw4 dib pr1 pr3-ns"></li></ul></div></div></nav></div></header><main class=pb7 role=main><div class="flex-l mt2 mw8 center"><article class="center cf pv5 ph3 ph4-ns mw7"><header><p class="f6 b helvetica tracked"></p><h1 class="f1-ns f4">Research</h1></header><div class="nested-copy-line-height lh-copy f4-ns f6 nested-links nested-img mid-gray"><h3 id=staphylococcus-aureus-mrsa><em>Staphylococcus aureus</em> (MRSA)</h3><p>The Gram-positive bacterium <em>S. aureus</em> is one of the leading causes of hospital-acquired infection in the US. Research projects have charactized the genomic epidemiology of clinical isolates and used genome-wide association approaches to understand the genetic basis of antibiotic-resistance, toxin production and phage resistance. We have also developed the “Staphopia” database of whole genome variation in more than 40,000 publically-available sequenced strains.</p><h3 id=chlamydia><em>Chlamydia</em></h3><p>Bacteria of the <em>Chlamydia</em> genus cause a wide range of infections in humans and animals. In particular, <em>C. trachomatis</em> is an extremely common sexually-transmitted infection across the globe. Our group has performed extensive genomic epidemiology to understand genetic determinants of virulences and track bacterial movement between human body sites.</p><h3 id=metagenomics>Metagenomics</h3><p>The EMERGENT group is involved in both 16S and shotgun metagenomics studies to understand correlations between microbiome composition and human health. We are also interested in using metagenome data for pathogen and antibiotic-resistance diagnostics and cross-referencing comparative genomics results. Dr Read is on the advisory board of the <a href=http://metasub.org>MetaSUB urban microbiome project</a> and we have collected environmental samples from the Atlanta subway as part of this project.</p><h3 id=other-projects>Other Projects</h3><p>Through a network of collaborations we have studied the genomes of a wide range of bacterial and fungal species and the whale shark (<em>Rhincodon typus</em>). Several projects have had a underlying forensics/ diagnotics theme, including analysis of plant/ chloroplast DNA in pollen mixtures.</p><h3 id=links>Links</h3><p>Our <a href=https://staphopia.emory.edu>Staphopia</a> database for assembling and calling SNPs in public <em>S. aureus</em> whole genome sequencing project</p><p>Our <a href=http://read-lab-confederation.github.io/nyc-subway-anthrax-study/>analysis</a> of putative anthrax detection in the NY subway system</p></div></article></div></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://read-lab-confederation.github.io>&copy; 2026 EMERGENT</a><div><a class=resp-sharing-button__link href="https://bsky.app/intent/compose?text=Check out EMERGENT, a bacterial genomics and metagenomics blog from Timothy Read's group! https://read-lab-confederation.github.io" target=_blank rel=noopener aria-label="Share on BlueSky"><div class="resp-sharing-button resp-sharing-button--bluesky resp-sharing-button--large"><div aria-hidden=true class="resp-sharing-button__icon resp-sharing-button__icon--solid"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 10.8c-1.087-2.114-4.046-6.053-6.798-7.995C2.566.944 1.561 1.266.902 1.565.139 1.908.0 3.08.0 3.768c0 .69.378 5.65.624 6.479.815 2.736 3.713 3.66 6.383 3.364.136-.02.275-.039.415-.056-.138.022-.276.04-.415.056-3.912.58-7.387 2.005-2.83 7.078 5.013 5.19 6.87-1.113 7.823-4.308.953 3.195 2.05 9.271 7.733 4.308 4.267-4.308 1.172-6.498-2.74-7.078a8.741 8.741.0 01-.415-.056c.14.017.279.036.415.056 2.67.297 5.568-.628 6.383-3.364.246-.828.624-5.79.624-6.478.0-.69-.139-1.861-.902-2.206-.659-.298-1.664-.62-4.3 1.24C16.046 4.748 13.087 8.687 12 10.8z"/></svg></div>Share on BlueSky</div></a></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script><script src=/js/custom.js></script></body></html>