---
title: "Research"
date: 2019-09-09T11:56:20-04:00
draft: false
---

### *Staphylococcus aureus* (MRSA)
The Gram-positive bacterium *S. aureus* is one of the leading causes of hospital-acquired infection in the US.  Research projects have charactized the genomic epidemiology of clinical isolates and used genome-wide association approaches to understand the genetic basis of antibiotic-resistance, toxin production and phage resistance.  


### *Chlamydia*
Bacteria of the *Chlamydia* genus cause a wide range of infections in humans and animals. In particular, *C. trachomatis* is an extremely common sexually-transmitted infection across the globe.  Our group has performed extensive genomic epidemiology to understand genetic determinants of virulences and track bacterial movement between human body sites.

### Metagenomics
The EMERGENT group is involved in several shotgun metagenomics studies to understand correlations between microbiome composition and human health.  We are also interested in using metagenome data for pathogen and antibiotic-resistance diagnostics and cross-referencing comparative genomics results. 

### Other Projects
Through a network of collaborations we have studied the genomes of a wide range of bacterial and fungal species and the whale shark (*Rhincodon typus*). Several projects have had a underlying forensics/ diagnotics theme, including analysis of plant/ chloroplast DNA in pollen mixtures.

### Recent Publications

#### 2025

Talbot BM, Jacko NF, Hofstetter KS, et al. Methicillin-Resistant Staphylococcus aureus Bacteremia Relapses Show Diverse Genomic Profiles but Convergence in Bacteremia-Associated Genes. *J Infect Dis.* 2025;232(6):1338-1350. [https://doi.org/10.1093/infdis/jiaf352](https://doi.org/10.1093/infdis/jiaf352)

LaRock DL, Sherman JD, Qu C, et al. Staphylococcus aureus induces Gasdermin A-dependent keratinocyte pyroptosis. *Nat Commun.* 2025;16(1):10570. [https://doi.org/10.1038/s41467-025-65674-3](https://doi.org/10.1038/s41467-025-65674-3)

Olagoke O, Chittaranjan S, Dean D. Transcriptional profiling of Chlamydia trachomatis and its host in an ex vivo endocervical primary cell culture system using dual RNA sequencing. *Front Cell Infect Microbiol.* 2025;15:1613922. [https://doi.org/10.3389/fcimb.2025.1613922](https://doi.org/10.3389/fcimb.2025.1613922)

Auchus IC, Brown J, Kama M, et al. Enhanced Chlamydia trachomatis and Neisseria gonorrhoeae Sexually Transmitted Infections and Associated Risk Factors in Fiji Following the Coronavirus Disease 2019 Pandemic. *Open Forum Infect Dis.* 2025;12(3):ofaf101. [https://doi.org/10.1093/ofid/ofaf101](https://doi.org/10.1093/ofid/ofaf101)

Olagoke O, Aziz A, Zhu LH, et al. Whole-genome automated assembly pipeline for Chlamydia trachomatis strains from reference, in vitro and clinical samples using the integrated CtGAP pipeline. *NAR Genom Bioinform.* 2025;7(1):lqae187. [https://doi.org/10.1093/nargab/lqae187](https://doi.org/10.1093/nargab/lqae187)

Bixler BJ, Royer CJ, Petit RA 3rd, Moller AG, Sefton S, Thomas S, Tunali A, Collins LF, Farley MM, Satola SW, Read TD. Comparative genomic analysis of emerging non-typeable Haemophilus influenzae (NTHi) causing emerging septic arthritis in Atlanta. *PeerJ.* 2025;13:e19081. [https://doi.org/10.7717/peerj.19081](https://doi.org/10.7717/peerj.19081)

Bommana S, Olagoke O, Hu YJ, Wang R, Kama M, Dehdashti M, Kodimerla R, Read TD, Dean D. Azithromycin alters the microbiome composition, function and resistome in women with Chlamydia trachomatis infections. *npj Biofilms and Microbiomes.* 2025;11:11. [https://doi.org/10.1038/s41522-025-00858-9](https://doi.org/10.1038/s41522-025-00858-9)

#### 2024

Hoang KL, Read TD, King KC. Defense Heterogeneity in Host Populations Gives Rise to Pathogen Diversity. *Am Nat.* 2024;204(4):370-380. [https://doi.org/10.1086/731996](https://doi.org/10.1086/731996)

Seth-Smith H, Bommana S, Dean D, et al. Chlamydia suis undergoes interclade recombination promoting Tet-island exchange. *BMC Genomics.* 2024;25(1):724. [https://doi.org/10.1186/s12864-024-10606-6](https://doi.org/10.1186/s12864-024-10606-6)

Raghuram V, Petit RA 3rd, Karol Z, et al. Average nucleotide identity-based Staphylococcus aureus strain grouping allows identification of strain-specific genes in the pangenome. *mSystems.* 2024;9(7):e0014324. [https://doi.org/10.1128/msystems.00143-24](https://doi.org/10.1128/msystems.00143-24)

Talbot BM, Clennon JA, Rakotoarison MFN, Rautman L, Durry S, Ragazzo LJ, Wright PC, Gillespie TR, Read TD. Metagenome-wide characterization of shared antimicrobial resistance genes in sympatric people and lemurs in rural Madagascar. *PeerJ.* 2024;12:e17805. [https://doi.org/10.7717/peerj.17805](https://doi.org/10.7717/peerj.17805)

Berasategui A, Salem H, Moller AG, et al. Genomic insights into the evolution of secondary metabolism of Escovopsis and its allies, specialized fungal symbionts of fungus-farming ants. *mSystems.* 2024;9(7):e0057624. [https://doi.org/10.1128/msystems.00576-24](https://doi.org/10.1128/msystems.00576-24)

Hofstetter KS, Jacko NF, Shumaker MJ, et al. Strain Differences in Bloodstream and Skin Infection: Methicillin-Resistant Staphylococcus aureus Isolated in 2018-2021 in a Single Health System. *Open Forum Infect Dis.* 2024;11(6):ofae261. [https://doi.org/10.1093/ofid/ofae261](https://doi.org/10.1093/ofid/ofae261)

Balthazar JT, Golparian D, Unemo M, et al. A laboratory-based predictive pathway for the development of Neisseria gonorrhoeae high-level resistance to corallopyronin A, an inhibitor of bacterial RNA polymerase. *Microbiol Spectr.* 2024;12(6):e0056024. [https://doi.org/10.1128/spectrum.00560-24](https://doi.org/10.1128/spectrum.00560-24)

Hoang KL, Read TD, King KC. Incomplete immunity in a natural animal-microbiota interaction selects for higher pathogen virulence. *Curr Biol.* 2024;34(6):1357-1363.e3. [https://doi.org/10.1016/j.cub.2024.02.015](https://doi.org/10.1016/j.cub.2024.02.015)

Alexander AM, Luu JM, Raghuram V, et al. Experimentally evolved Staphylococcus aureus shows increased survival in the presence of Pseudomonas aeruginosa by acquiring mutations in the amino acid transporter, GltT. *Microbiology (Reading).* 2024;170(3). [https://doi.org/10.1099/mic.0.001445](https://doi.org/10.1099/mic.0.001445)

Bouiller K, Jacko NF, Shumaker MJ, et al. Factors associated with foreign body infection in methicillin-resistant Staphylococcus aureus bacteremia. *Front Immunol.* 2024;15:1335867. [https://doi.org/10.3389/fimmu.2024.1335867](https://doi.org/10.3389/fimmu.2024.1335867)

Babiker A, Karadkhele G, Bombin A, et al. The Burden and Impact of Early Post-transplant Multidrug-Resistant Organism Detection Among Renal Transplant Recipients, 2005-2021. *Open Forum Infect Dis.* 2024;11(3):ofae060. [https://doi.org/10.1093/ofid/ofae060](https://doi.org/10.1093/ofid/ofae060)

Vanderwoude J, Azimi S, Read TD, Diggle SP. The role of hypermutation and collateral sensitivity in antimicrobial resistance diversity of Pseudomonas aeruginosa populations in cystic fibrosis lung infection. *mBio.* 2024;15(2):e0310923. [https://doi.org/10.1128/mbio.03109-23](https://doi.org/10.1128/mbio.03109-23)

Bommana S, Hu Y-J, Kama M, et al. Unique microbial diversity, community composition, and networks among Pacific Islander endocervical and vaginal microbiomes with and without Chlamydia trachomatis infection in Fiji. *mBio.* 2024;15(1):e0306323. [https://doi.org/10.1128/mbio.03063-23](https://doi.org/10.1128/mbio.03063-23)

#### 2023

Raghuram V, Gunoskey JJ, Hofstetter KS, et al. Comparison of genomic diversity between single and pooled Staphylococcus aureus colonies isolated from human colonization cultures. *Microb Genom.* 2023;9(11). [https://doi.org/10.1099/mgen.0.001111](https://doi.org/10.1099/mgen.0.001111)

Yue Y, Read TD, Fedirko V, et al. Integrative analysis of microbial 16S gene and shotgun metagenomic sequencing data improves statistical efficiency. *Res Sq.* 2023. [https://doi.org/10.21203/rs.3.rs-3376801/v1](https://doi.org/10.21203/rs.3.rs-3376801/v1)

Holley CL, Dhulipala V, Maurakis SA, et al. Transcriptional activation of ompA in Neisseria gonorrhoeae mediated by the XRE family member protein NceR. *mBio.* 2023;14(4):e0124423. [https://doi.org/10.1128/mbio.01244-23](https://doi.org/10.1128/mbio.01244-23)

Mehta RS, Petit RA 3rd, Read TD, Weissman DB. Detecting patterns of accessory genome coevolution in Staphylococcus aureus using data from thousands of genomes. *BMC Bioinformatics.* 2023;24(1):243. [https://doi.org/10.1186/s12859-023-05363-4](https://doi.org/10.1186/s12859-023-05363-4)

Joseph SJ, Bommana S, Ziklo N, et al. Patterns of within-host spread of Chlamydia trachomatis between vagina, endocervix and rectum revealed by comparative genomic analysis. *Front Microbiol.* 2023;14:1154664. [https://doi.org/10.3389/fmicb.2023.1154664](https://doi.org/10.3389/fmicb.2023.1154664)

Auchus IC, Kama M, Bhuiyan RA, Brown J, Dean D. Chlamydial and gonorrheal neglected sexually transmitted diseases among Pacific Islanders of the Western Pacific Region-A narrative review and call to action. *PLoS Negl Trop Dis.* 2023;17(3):e0011171. [https://doi.org/10.1371/journal.pntd.0011171](https://doi.org/10.1371/journal.pntd.0011171)

Bianco CM, Moustafa AM, O'Brien K, et al. Pre-epidemic evolution of the MRSA USA300 clade and a molecular key for classification. *Front Cell Infect Microbiol.* 2023;13:1081070. [https://doi.org/10.3389/fcimb.2023.1081070](https://doi.org/10.3389/fcimb.2023.1081070)

Moller AG, Petit RA 3rd, Davis MH, Read TD. Development of an Amplicon Nanopore Sequencing Strategy for Detection of Mutations Conferring Intermediate Resistance to Vancomycin in Staphylococcus aureus Strains. *Microbiol Spectr.* 2023;11(1):e0272822. [https://doi.org/10.1128/spectrum.02728-22](https://doi.org/10.1128/spectrum.02728-22)


