---
title: "Genomics of MRSA evolution and persistence on the human body"
description: ""
author: ["Tim Read"]
date: 2019-09-10T06:45:13-04:00
featured_image: "/images/2019-USA300-background.png"
draft: false
---
[Read TD, Petit III RA, Yin Z, Montgomery T, McNulty MC, David MZ. USA300 Staphylococcus aureus persists on multiple body sites following an infection. BMC Microbiol [Internet]. 2018 Dec 5;18(1):206. Available from: https://doi.org/10.1186/s12866-018-1336-z](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-018-1336-z)

Microbiologists know surprisingly little about the dynamics of bacterial life on the human body: how strains are acquired, how they move between body sites and how long it takes for them to die off or be replaced. This is because, until recently, we didn’t have the genetic tools that allow us to say whether two bacteria from the same species shared a last common ancestor 2 months ago or 1000 years ago.  Affordable genome sequencing introduced over the past 10 years has changed the game and produced a raft of outstanding insights into evolution of bacteria that are on a single person’s body, or “within-host”<sup><a href="https://paperpile.com/c/BGdSfL/lpD0+6zf2+cADQ">1–3</a></sup>.  

At the end of last year I published the results of a productive collaboration with Michael David at U. Penn<sup><a href="https://paperpile.com/c/BGdSfL/zEql">4</a></sup> that gave us a glimpse into what happens to a notorious human pathogen (_Staphylococcus aureus_) after it has caused an acute infection.  Michael enrolled 30 patients with an infection caused by methicillin-resistant _S. aureus_ (MRSA) strain USA300. This was the first known MRSA infection that each of the subjects had ever had. Shortly after the initial isolate was taken from the site of infection, swab cultures were taken from three potential “colonization” sites - throat, nose and rectum - and we attempted to isolate MRSA.  A year later, the patients returned (most of them had shown no signs of infection recurrence) and the same three sites were swabbed.  All the initial infecting and colonizing MRSA were then genome sequenced and analyzed using our [Staphopia](https://staphopia.emory.edu) analysis pipeline software<sup><a href="https://paperpile.com/c/BGdSfL/EBJz">5</a></sup>.

All the USA300 genomes were very similar to each other because they all derive from the same bacterial ancestor at the start of the outbreak of community-acquired MRSA in the late 1980s<sup><a href="https://paperpile.com/c/BGdSfL/XB67">6</a></sup>. However, strains from the same patient tended to have many fewer genetic differences (single nucleotide polymorphisms: SNPs) between them than did strains from different patients. This cutoff of about 40 SNPs in the core genome alignment allowed us to define “intra-subject lineages” (ISLs) that colonized the patients before causing SSTI and persisted after the infection.  We can’t rule out that bacteria in ISLs are being lost from a person and then re-acquired from surfaces and other family members - we know from previous studies that households can be colonized by USA300 clones for many years<sup><a href="https://paperpile.com/c/BGdSfL/5ad2">7</a></sup> - but we presume that the simplest explanation is that bacteria in ISLs are persisting and spreading among sites on humans.  

![Figure 2 - intra/inter ISL](/images/2019-USA300-behind-the-paper0.png)

While this study was an observational pilot not powered to draw strong conclusions, it did provide some fascinating insights that we will be following up on.  We found that an USA300 ISL **lingers asymptomatically on patients after infection**, although the numbers of patients with the strain declined at the one year sampling point.  We also found that during their stay on patients strains **occasionally lost and gained mobile genetic elements** that changed the antibiotic resistance profile. **Persistence was not associated exclusively with any one of the three tested body sites,** and in many cases a strain appeared at a site on the second sampling that wasn't found in the first sampling, **suggesting movement between the sites**.  The latter results challenge the usually accepted model that the nose is the primary place where _S. aureus_ lives on the human body when it is not causing disease.

This paper (along with other within-host studies with Taj Azarian from University of Central Florida<sup><a href="https://paperpile.com/c/BGdSfL/f8NQ+cYdR">8,9</a></sup>) helped us to convince the NIH to give us an RO1 grant to look at MRSA colonization in much greater genetic detail on many more patients. The project, called SEMAPHORE (<span style="text-decoration:underline;">S</span>tudy of <span style="text-decoration:underline;">M</span>RSA, <span style="text-decoration:underline;">A</span>ntibiotics and <span style="text-decoration:underline;">P</span>ersistence, <span style="text-decoration:underline;">H</span>aving the <span style="text-decoration:underline;">O</span>utcome of <span style="text-decoration:underline;">Re</span>currence), is currently in the enrollment stage.  We aim to sequence many thousands of genomes to understand how *S. aureus* that is part of an ISL endures antibiotics, evolves and dies and whether we can determine the genetic factors that predispose a certain subset of patients to serious recurrent MRSA infections. 

#### References
1.  [Zhao S, Lieberman TD, Poyet M, Kauffman KM, Gibbons SM, Groussin M, Xavier RJ, Alm EJ. Adaptive Evolution within Gut Microbiomes of Healthy People. Cell Host Microbe [Internet]. 2019 Apr 23; Available from: http://www.sciencedirect.com/science/article/pii/S1931312819301593](http://paperpile.com/b/BGdSfL/lpD0)

2.  [Moradigaravand D, Gouliouris T, Blane B, Naydenova P, Ludden C, Crawley C, Brown NM, Török ME, Parkhill J, Peacock SJ. Within-host evolution of Enterococcus faecium during longitudinal carriage and transition to bloodstream infection in immunocompromised patients. Genome Med [Internet]. 2017 Dec 27;9(1):119. Available from: http://dx.doi.org/10.1186/s13073-017-0507-0 PMID: 29282103](http://paperpile.com/b/BGdSfL/6zf2)

3.  [Didelot X, Sarah Walker A, Peto TE, Crook DW, Wilson DJ. Within-host evolution of bacterial pathogens. Nat Rev Microbiol [Internet]. Nature Publishing Group; 2016 Jan 19 [cited 2016 Jan 24]; Available from: http://dx.doi.org/10.1038/nrmicro.2015.13](http://paperpile.com/b/BGdSfL/cADQ)

4.  [Read TD, Petit RA, Yin Z, Montgomery T, McNulty MC, David MZ. USA300 Staphylococcus aureus persists on multiple body sites following an infection. BMC Microbiol [Internet]. 2018 Dec 5;18(1):206. Available from: https://doi.org/10.1186/s12866-018-1336-z](http://paperpile.com/b/BGdSfL/zEql)

5.  [Petit RA 3rd, Read TD. Staphylococcus aureus viewed from the perspective of 40,000+ genomes. PeerJ [Internet]. 2018 Jul 12;6:e5261. Available from: http://dx.doi.org/10.7717/peerj.5261 PMCID: PMC6046195](http://paperpile.com/b/BGdSfL/EBJz)

6.  [David MZ, Daum RS. Community-associated methicillin-resistant Staphylococcus aureus: epidemiology and clinical consequences of an emerging epidemic. Clin Microbiol Rev [Internet]. 2010 Jul;23(3):616–687. Available from: http://dx.doi.org/10.1128/CMR.00081-09 PMCID: PMC2901661](http://paperpile.com/b/BGdSfL/XB67)

7.  [Alam MT, Read TD, Petit RA, Boyle-Vavra S, Miller LG, Eells SJ, Daum RS, David MZ. Transmission and Microevolution of USA300 MRSA in U.S. Households: Evidence from Whole-Genome Sequencing. MBio [Internet]. American Society for Microbiology; 2015 Mar 10;6(2):e00054–15. Available from: http://dx.doi.org/10.1128/mBio.00054-15 PMID: 25759497](http://paperpile.com/b/BGdSfL/5ad2)

8.  [Azarian T, Ridgway J, Yin Z, David M. Long-term intrahost evolution of methicillin resistant Staphylococcus aureus among cystic fibrosis patients with respiratory carriage. Front Genet [Internet]. 2019 [cited 2019 Apr 4];10:546. Available from: https://www.frontiersin.org/article/10.3389/fgene.2019.00546](http://paperpile.com/b/BGdSfL/f8NQ)

9.  [Azarian T, Daum RS, Petty LA, Steinbeck JL, Yin Z, Nolan D, Boyle-Vavra S, Hanage WP, Salemi M, David MZ. Intrahost Evolution of Methicillin-Resistant Staphylococcus aureus USA300 Among Individuals With Reoccurring Skin and Soft-Tissue Infections. J Infect Dis [Internet]. 2016 Sep 15;214(6):895–905. Available from: http://dx.doi.org/10.1093/infdis/jiw242 PMCID: PMC4996148](http://paperpile.com/b/BGdSfL/cYdR)

