---
title: "Meet the Staph: A menagerie of common *S. aureus* reference strains" 
description: ""
author: ["Brooke Talbot","Vishnu Raghuram"]
date: "2022-10-31"
featured_image: "/images/blog202210_Image1.jpg"
draft: false
---

When we want to answer a specific phenotype question about bacteria, we are often left to ask which strain is the best choice? It’s important to have a variety of strain backgrounds for your laboratory experiments because different genetic backgrounds will help to answer different questions. This extends to all manner of living laboratory organisms from *C. elegans* to larger vertebrates. In the context of human health research, these higher order model organisms have been bred and manipulated to optimize their specific relevance to modeling a human health concern. Similarly, bacteria can also be modified to adapt to a specific laboratory environment to answer important questions. In contrast, these pathogenic bacterial strains already have important clinical, ecological, and epidemiological contexts which may have contributed to their unique phenotypes and their genetic profiles.

## *Staphylococcus aureus* on parade
One particular organism with a long laboratory history is *Staphylococcus aureus*, a favorite focal species in the EMERGENT group. Multidrug resistant *S. aureus*, such as methicillin resistant *S. aureus* (MRSA), are a global priority pathogen targeted for decreasing morbidity and mortality in humans, particularly in regards to its impact on hospitalized individuals and those in long-term care facilities ([WHO 2017](https://www.who.int/publications/i/item/WHO-EMP-IAU-2017.12)). Additionally, Some *S. aureus* infections are agriculturally-associated, and transmission from a carrier to food items can result in foodborne illness. All of this together demonstrates the epidemiological versatility of *S. aureus* and the need to foster research focusing on different contexts to prevent future illnesses.

Well established strains of *S. aureus* which have been cataloged, propagated, and utilized go as far back as the 1950s, prior even to the discovery of the structure of DNA. Strains Newman, NCTC8352, and N315 were being used as laboratory models to better understand the biology of *S. aureus* for decades, all while a better understanding of nucleic acid sequencing was itself evolving. The emergence of “next generation” shotgun short read sequencing ushered in an important era for more complex and complete genomic analyses ([Heather and Chain 2016](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4727787/)), bringing new context and exploratory studies of the genetic basis for pathogen emergence. 

![Fig1](/images/blog202210_Image1.jpg "Fig1")

We identified seven *S. aureus* strains that are commonly used in laboratory work exploring *S. aureus* biology, particularly for cloning experiments and experimental evolution, and also occasionally as the source of reference sequences. If you are in the world of *S. aureus*, you have likely heard these names, but did you know all of their biography? If you're new to this space, or looking for more context for your genomic work on *S. aureus*, then allow us to introduce you to some of the key microbial players in field. We have noted these strains biological source, year of isolation, year of complete sequencing, and also tried to understand why they’re named what they’re named, if possible. We also provided a quick highlight about the unique utility of these strains for research questions.


## JE2 (USA300 LAC): 
- Biological source: Skin abscess of an individual from Los Angeles county jail
- Sequence Type 8, Clonal Complex 8
- Isolate citation:[CDC 2003](https://www.cdc.gov/mmwr/preview/mmwrhtml/mm5205a4.htm)
- Complete sequence citation:[Diep 2006](https://doi.org/10.1016/S0140-6736(06)68231-7)
- Year isolated: ~2003
- Year completely sequenced: 2006 
- Source of name: Maybe author initials (Jennifer Endres)
- Highlight: Nebraska transposon library
- Chromosome NCBI Reference Sequence: [NZ_CP020619.1](https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP020619.1?report=graph)

The USA300 lineage is the most common community associated  MRSA (CA-MRSA) lineage circulating in the US. The USA300 reference strain JE2 is a derivative of USA300 LAC, which is a skin infection isolate obtained from an outbreak in the Los Angeles County jail, hence the name. The original USA300 LAC isolate was cured of plasmids to facilitate easier genetic manipulation, leading to the isolate JE2. JE2 was used as the backbone for the construction of the [Nebraska Transposon Mutant Library (NTML)](https://doi.org/10.1128/mBio.00537) by Fey et al in 2003, which has grown to become a familiar and important resource for *S. aureus* scientists. We assume the ‘JE’ in JE2 correspond to the initials of one of the authors in the NTML paper, Dr. Jennifer L. Endres, and 2 just happens to be the designation for the colony that was confirmed to be plasmid cured. Due to the major presence of USA300 lineages in the US, and the widespread use of the NTML, JE2 remains one of the most common and accessible isolates of *S. aureus*. Strain FPR3757, an isolate with an identical core genome to JE2, is used as the reference genome sequence. 

## N315:
- Biological source: “Hospitals in Japan”
- Sequence Type 5, Clonal Complex 5 
- Isolate citation: [Okonogi 1989](https://doi.org/10.1093/jac/24.5.637)
- Complete sequence citation: [Kuroda 2001](https://doi.org/10.1016/S0140-6736(00)04403-2) 
- Year isolated: 1982
- Year completely sequenced: 2001
- Source of name: Nagasaki University isolate no. 315
- Highlight: First strain where *mecA* was cloned and sequenced
- Chromosome NCBI Reference Sequence: [NC_002745.2](https://www.ncbi.nlm.nih.gov/nuccore/NC_002745.2?report=graph)
- Plasmid pN315 NCBI Reference Sequence: [NC_003140.1](https://www.ncbi.nlm.nih.gov/nuccore/NC_003140.1?report=graph)

We have not found any information about the origin of N315 other than that it was isolated from a patient in a Japanese hospital in 1982. It was first reported in a study in 1989 by Okonogi et al. The authors compared *S. aureus* isolates from the early 1970s and after 1982 to investigate the rapid emergence of MRSA in Japan in the 80s. Though N315 carries *mecA*, it was found to be susceptible to methicillin in laboratory conditions. This eventually led to the discovery of the repressor *mecI* and the antirepressor *mecR1*, which collectively make up the 3 component regulatory system controlling methicillin resistance in *S. aureus*. The first molecular cloning of *mecA* was performed in N315, as well as the first complete sequence of the SCCmec cassette. N315 has since been a commonly used prototypical MRSA strain that led to several studies regarding regulation and characterization of *mecA*. Interestingly, N315 is deficient in *agr*, a key virulence regulator in *S. aureus* due to a truncation in the *agrA* gene. This makes N315 a good candidate for studying antimicrobial resistance, but not for *agr* mediated virulence regulation in *S. aureus*.

## MW2: 
- Biological source:  Isolate from fatal septicemia and arthritis of a 16 month-old girl from North Dakota, USA,
 - Sequence Type 1, Clonal Complex 1
- Isolate citation: [Hunt 1999](https://jamanetwork.com/journals/jamadermatology/fullarticle/478114)
- Complete sequence citation: [Baba 2002](https://doi.org/10.1016/S0140-6736(02)08713-5)
- Year isolated: 1998
- Year completely sequenced: 2002
- Source of name: “Midwest 2”  
- Highlight: Community associated and community acquired (CA), high virulence and multidrug resistant including beta-lactams
- Chromosome NCBI Reference Sequence: [NC_003923.1](https://www.ncbi.nlm.nih.gov/nuccore/NC_003923.1?report=graph)

MW2 (Midwest-2) is a CA-MRSA strain with a pulse-field gel electrophoresis genotype similar to strains commonly reported in the Midwest, USA. MW2 was initially reported in a CDC report detailing pediatric deaths from CA-MRSA strains in North Dakota and Minnesota between 1997 and 1999. MW2 makes a sobering case for respecting and acknowledging the context behind which we have obtained the tools to carry out our science. MW2 was isolated from an infant who died within 2 hrs of arriving at the hospital, after suffering from seizures, respiratory failure and cardiac arrest. The patient had no known risk factors and no known close-contacts who visited health-care settings in the past year. This was also during a time when reports of MRSA infections were rapidly increasing, however MRSA was usually thought to spread through hospital acquisition and not through community. MW2 is now a prototypical CA-MRSA strain commonly used in skin-infection and abscess models, contributing greatly to our understanding of how *Staphylococcal* efflux pumps can aid in colonization of the skin.

## NCTC 8325
- Biological source: Corneal ulcer
- Sequence Type 8, Clonal Complex 8
- Isolate citation: [Novick 1967](https://doi.org/10.1016/0042-6822(67)90105-5)
- Complete sequence citation:[Gillaspy 2006](https://doi.org/10.1128/9781555816513.ch32)
- Year isolated: ~1950  
- Year completely sequenced: 2000
- Source of name: NCTC submission number 
- Highlight: First *S. aureus* strain to be sequenced
- Chromosome NCBI Reference Sequence: [NC_007795.1](https://www.ncbi.nlm.nih.gov/nuccore/NC_007795.1?report=graph)

NCTC or “National Collection of Type Cultures” was established in 1920 by the UK health security agency and is the oldest known culture-collection center in the world. *S. aureus* strain 8325 was isolated ~ 1950 from a corneal ulcer from the Central Public Health Laboratory in Colindale, London [Novick, 1967](https://doi.org/10.1016/0042-6822(67)90105-5). While this isolate was used initially for phage propagation, 8325 (and its derivatives) went on to become a major model of virulence regulation in *S. aureus*. 8325 is also designated RN1, or “Richard Novick 1” as it was initially introduced by the Novick lab in 1967, followed by several studies by the Novick lab and others characterizing *Staphylococcal* virulence, antibiotic resistance, phage sensitivity, and global gene regulation. RN1 is deemed the “the *Staphylococcal* counterpart of *Escherichia coli* K-12” [Herbert, 2010](https://doi.org/10.1128/IAI.00088) due to this isolate being the backbone for several fundamental discoveries about *S. aureus* biology we take for granted today. 

## RN4220
- Biological source: Lab generated: UV treatment of 8325 to cure phages followed by chemical mutagenesis to inactivate restriction system
- Sequence Type 8, Clonal Complex 8
- Isolate citation: [Kreiswirth 1983](https://www.nature.com/articles/305709a0)
- Complete sequence citation [Nair 2011](https://doi.org/10.1128/JB.00027-11)
- Year isolated generated: 1983
- Year completely sequenced: 2011
- Source of name: Richard Novick 4220
- Highlight: No restriction barrier
- Chromosome NCBI Reference Sequence: [NZ_CP076105.1](https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP076105.1?report=graph)

Due to a stringent restriction system that limits uptake of non-staphylococcal DNA, performing genetic manipulation in *S. aureus* is 
a difficult task. Enter RN4220, an important derivative of the aforementioned NCTC 8325. RN4220 is a phage-cured, restriction enzyme 
negative version of 8325, making it a prime candidate for genetic manipulation in *S. aureus*. RN4220 is commonly used as an 
intermediate host for foreign DNA, after which the DNA fragment is typically transduced into the isolate of interest. It is important 
to note that while the parent strain 8325 is used as a virulence model, RN4220 is defective in the primary *Staphylococcal* virulence 
regulator *agr* presumably due to SNPs introduced during the mutagenesis process making it incompatible for virulence studies. 
Nevertheless, it is still widely used as an intermediate cloning host to this day. 

## Newman (NCTC 8178): 
- Biological source: tubercular osteomyelitis isolate from a hospitalized patient
- Sequence Type 254, Clonal Complex 8
- Isolate citation: [Duthie and Lorenz 1952](https://pubmed.ncbi.nlm.nih.gov/14927856/)
- Complete sequence citation: [Baba 2008](https://pubmed.ncbi.nlm.nih.gov/17951380/)
- Year Isolated: 1952
- Year completely sequenced: 2008
- Source of name: Unknown
- Highlight: A methicillin-susceptible strain with high virulence
- Chromosome NCBI Reference Sequence: [NC_009641.1](https://www.ncbi.nlm.nih.gov/nuccore/NC_009641.1?report=graph)

Attribution and authorship of this strain record belong to Dr. E.S. Dulthie [citation], though the actual isolates used in this publication originated from a collection of clinical lab isolates from Charles H. Lack at the Royal National Orthopaedic Hospital, in Stalmore, UK, who also appeared to be a dear friend of Dulthie([Lack 1959](https://doi-org.proxy.library.emory.edu/10.1038/184589a0)). The NCTC already existed for reference strains at this time, with at least one *S. aureus* strain [available in 1944](https://www.culturecollections.org.uk/products/bacteria/detail.jsp?refId=NCTC%206571&collection=nctc), but Dulthie’s experimental purposes for strain selection was to identify a strain phenotype with the most optimal coagulase activity, and it being a strong positive warranted the continuation of its use experimentally and subsequent archival as NCTC 8178. Despite our best efforts, we could not find out why this strain is called “Newman.” It does not appear to be associated with the name of a collaborator or co-author.  
Despite its early isolation, the complete genome was not available until 2008. The sequence showed that Newman has 2 pathogenicity islands and four distinct prophages that contribute greatly to its virulence([Baba 2008](https://journals-asm-org.proxy.library.emory.edu/doi/10.1128/JB.01000-07)). Newman is genetically close to JE2, remaining methicillin-susceptible and virulent toward humans, and having clear markers of virulence apparent in these prophage regions. It therefore serves as an interesting reference and outgroup for comparative evolution studies among community-associated strains acquired in a post-methicillin-resistant age.    

## Mu50 (VRSA)

- Biological Source: Pus of a wound infection from a male infant
- Sequence Type 5, Clonal Complex 5
- Year isolated: 1997 
- Isolate citation:  ([Hiramatsu 1997](https://pubmed.ncbi.nlm.nih.gov/9249217/))
- Complete sequence citation: ([Kuroda 2001](https://pubmed.ncbi.nlm.nih.gov/11418146/))
- Source of name: Unknown
- Highlights: Provides a sequence for vancomycin resistant *S. aureus*
- Chromosome NCBI Reference Sequence:[NC_002758.2](https://www.ncbi.nlm.nih.gov/nuccore/NC_002758.2?report=graph)
- Plasmid VRSAp NCBI Reference Sequence: [NC_002774.1](https://www.ncbi.nlm.nih.gov/nuccore/NC_002774.1?report=graph)

The Mu50 strain was isolated in 1997 from the pus of a surgical site infection of a male infant patient in Japan ([Hiramatsu 1997](https://pubmed.ncbi.nlm.nih.gov/9249217/)), and four years later its genome was sequenced ([Kuroda 2001](https://pubmed.ncbi.nlm.nih.gov/11418146/)). A decade before its sequencing, in 1991, injectable Vancomycin was made available as a course of treatment for MRSA in Japan. However, similar to many stories of antibiotic resistance in bacteria, it did not take long for vancomycin-resistant strains to emerge and spread amongst Japanese hospitalized patients. Interestingly, though there was concern of plasmid-associated genes *vanA* and *vanB* conferring vancomycin resistance in *Enterococcus* species, the Mu50 strain does not contain either of these elements, indicating some other mechanism for the increased resistance to vancomycin. The relatively time interval between isolation and sequencing reflect the growing motivation to apply genomics to clinical context, and the advent of an increased understanding and availability of sequencing information. The impending genomic analyses of Mu50, when compared to the non-vancomycin resistant MRSA strain N315, revealed curious but not outright obvious differences in the ORFs, including loss-of-function mutations related to growth, particularly MutS, as well as  peptidoglycan synthesis and production of glutamine ([Avison 2002]( https://academic.oup.com/jac/article/49/2/255/781426 2002)). Mu50 has a much thicker cell wall than other resistant MRSA strains, and that it carries loss of function mutations. In a laboratory setting, it may be useful to see if there are common phenotypic presentations of vancomycin resistance when different genetic backgrounds are compared. It is not useful if trying to understand the evolution of phenotypic vancomycin resistance in the context of understanding genes that have been directly associated with resistance (*vanA* and *vanB*). Evidence of naturally occurring acquisition of *vanA* and *vanB* genes in *S. aureus* ([Maharjan 2021](https://pubmed.ncbi.nlm.nih.gov/34899917/), [Kalasseril Girijan and Pillai 2021](https://pubmed-ncbi-nlm-nih-gov.proxy.library.emory.edu/34665771/)) necessitates additional representation of a VRSA reference strain containing this genetic context.  

## A short list for a big family

Common reference strains cited in literature both for laboratory usage and for genomic sequencing only fall into a few of these lineages, particularly Clonal complexes (CC) 5 and 8. 

![Fig2](/images/blog202210_Image2.png "Fig2")

However, PubMLST([Jolley 2014](https://pubmed.ncbi.nlm.nih.gov/30345391/)) documents the [10 most common globally distributed and unambiguously assigned CCs](https://pubmlst.org/organisms/staphylococcus-aureus/clonal-complexes), which includes: CC1, 5, 8, 15, 22, 30, 45, 97, 93, 121. The divide between different *S. aureus* clonal lineages is interesting as there are distinct genomic differences between CC, including lineage-associated mobile genetic elements (eg. SCCmec) and virulence factors (eg. PVL), leading to unique resistance and virulence phenotypes. There are also unique ecological patterns among different backgrounds of MRSA, including healthcare associations and livestock associations. 

The bias in reference strains could arise from the pervasiveness of usage of these particular strains prior to the development of Next Generation sequencing, and because these strains have been so well characterized phenotypically, it makes sense to start with these strains to understand genomic patterns and genetic associations. However, It is important to recognize that a genomic question may be related but different from a laboratory question. The strength of a genomic approach is that through comparison of sequences, one can extract the most identifying information about strains gathered through surveillance or natural experiments, including the prevalence of a gene, the evolutionary history of a strain, and the spread of mobile genetic elements between species. It is therefore important to find a reference sequence that limits gaps in alignments and has comparable homologous regions by which variation can be detected rather than thrown away. In contrast, a laboratory-based approach may want to focus on a reference that has the optimal phenotypic characteristics for testing a hypothesis. Increasingly, it feels like it is becoming more and more important to conduct research from both sides which offers a more robust understanding of strain-associated characteristics for the ecology and evolution of *S. aureus*. 

## Strains come from somewhere (or from someone)

These listed strains are important for understanding basic bacterial processes, identifying causes and potential targets for preventing illness, and unraveling complicated phenotype-genotype relationships. Nevertheless, these common strains may not be biologically representative to answer all questions, and it begs the question of whether genomic approaches focusing on a single reference for large-scale analyses is an appropriate approach. Development of a standardized pan-genome, as well as better characterization and sampling of strains in areas with historically limited sampling efforts would help refine and enrich the current standard, curated list of well-cited reference strains. 

In this post, we have primarily touched on the idea that strain choice depends on the best biological match for the research question. In the process, we were able to dig into particular historical and personal nuances of the reference sequences including the origin of strain names (some more successfully than others, despite hounding emails to various medical archives!), the congenial sharing of isolates between friends and colleagues, and the harrowing stories of patient encounters by which clinicians and scientists came to acquire a clinical sample.  We know the clinical context, and we have many of these strains because someone somewhere was ill and suffered. With these *S. aureus* strains we also happen to know that many of these strains came from patients in vulnerable populations. It’s wise, and ethically fundamental, to ask ourselves an additional question during the research development phase: By using this strain, how does my work contribute to justice for the patient, environment, or community from which it was derived? To answer this question, it will take some careful thought in the design phase of any research, but will be a worthwhile step in ensuring a project practices beneficence and justice. 
