<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>EMERGENT | Genomics of MRSA evolution and persistence on the human body</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.79.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css rel=stylesheet><link rel=stylesheet href=/css/custom.css><meta property="og:title" content="Genomics of MRSA evolution and persistence on the human body"><meta property="og:description" content="Read TD, Petit III RA, Yin Z, Montgomery T, McNulty MC, David MZ. USA300 Staphylococcus aureus persists on multiple body sites following an infection. BMC Microbiol [Internet]. 2018 Dec 5;18(1):206. Available from: https://doi.org/10.1186/s12866-018-1336-z
Microbiologists know surprisingly little about the dynamics of bacterial life on the human body: how strains are acquired, how they move between body sites and how long it takes for them to die off or be replaced. This is because, until recently, we didn’t have the genetic tools that allow us to say whether two bacteria from the same species shared a last common ancestor 2 months ago or 1000 years ago."><meta property="og:type" content="article"><meta property="og:url" content="https://read-lab-confederation.github.io/blog/posts/mrsa-evolution-persistence/"><meta property="article:published_time" content="2019-09-10T06:45:13-04:00"><meta property="article:modified_time" content="2019-09-10T06:45:13-04:00"><meta itemprop=name content="Genomics of MRSA evolution and persistence on the human body"><meta itemprop=description content="Read TD, Petit III RA, Yin Z, Montgomery T, McNulty MC, David MZ. USA300 Staphylococcus aureus persists on multiple body sites following an infection. BMC Microbiol [Internet]. 2018 Dec 5;18(1):206. Available from: https://doi.org/10.1186/s12866-018-1336-z
Microbiologists know surprisingly little about the dynamics of bacterial life on the human body: how strains are acquired, how they move between body sites and how long it takes for them to die off or be replaced. This is because, until recently, we didn’t have the genetic tools that allow us to say whether two bacteria from the same species shared a last common ancestor 2 months ago or 1000 years ago."><meta itemprop=datePublished content="2019-09-10T06:45:13-04:00"><meta itemprop=dateModified content="2019-09-10T06:45:13-04:00"><meta itemprop=wordCount content="1004"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="Genomics of MRSA evolution and persistence on the human body"><meta name=twitter:description content="Read TD, Petit III RA, Yin Z, Montgomery T, McNulty MC, David MZ. USA300 Staphylococcus aureus persists on multiple body sites following an infection. BMC Microbiol [Internet]. 2018 Dec 5;18(1):206. Available from: https://doi.org/10.1186/s12866-018-1336-z
Microbiologists know surprisingly little about the dynamics of bacterial life on the human body: how strains are acquired, how they move between body sites and how long it takes for them to die off or be replaced. This is because, until recently, we didn’t have the genetic tools that allow us to say whether two bacteria from the same species shared a last common ancestor 2 months ago or 1000 years ago."><script type=application/javascript>var doNotTrack=false;if(!doNotTrack){window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;ga('create','UA-148959477-1','auto');ga('send','pageview');}</script><script async src=https://www.google-analytics.com/analytics.js></script><link href=/css/share-button.css rel=stylesheet></head><body class="ma0 avenir bg-near-white"><header class="cover bg-top" style=background-image:url(https://read-lab-confederation.github.io/images/2019-USA300-background.png)><div class="pb3-m pb6-l bg-black-60"><nav class="pv3 ph3 ph4-ns tc tl-l" role=navigation><div class="flex-l justify-between items-center center"><a href=https://read-lab-confederation.github.io class="f3 fw2 hover-white no-underline white-90 dib">EMERGENT</a><div class="flex-l items-center"><ul class="pl0 mr3"><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/blog/ title="Blog page">Blog</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/group/ title="Group page">Group</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/research/ title="Research page">Research</a></li><li class="list f5 f4-ns fw4 dib pr1 pr3-ns"><a href=https://twitter.com/tdread_emory target=_blank class="link-transition twitter link dib z-999 pt0-l mr1" title="Twitter link" rel=noopener aria-label="follow on Twitter——Opens in a new window"><svg height="24" style="enable-background:new 0 0 67 67" viewBox="0 0 67 67" width="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M37.167 22.283c-2.619.953-4.274 3.411-4.086 6.101l.063 1.038-1.048-.127c-3.813-.487-7.145-2.139-9.974-4.915l-1.383-1.377-.356 1.017c-.754 2.267-.272 4.661 1.299 6.271.838.89.649 1.017-.796.487-.503-.169-.943-.296-.985-.233-.146.149.356 2.076.754 2.839.545 1.06 1.655 2.097 2.871 2.712l1.027.487-1.215.021c-1.173.0-1.215.021-1.089.467.419 1.377 2.074 2.839 3.918 3.475l1.299.444-1.131.678c-1.676.976-3.646 1.526-5.616 1.568C19.775 43.256 19 43.341 19 43.405c0 .211 2.557 1.397 4.044 1.864 4.463 1.377 9.765.783 13.746-1.568 2.829-1.673 5.657-5 6.978-8.221.713-1.716 1.425-4.851 1.425-6.354.0-.975.063-1.102 1.236-2.267.692-.678 1.341-1.419 1.467-1.631.21-.403.188-.403-.88-.043-1.781.636-2.033.551-1.152-.402.649-.678 1.425-1.907 1.425-2.267.0-.063-.314.042-.671.233-.377.212-1.215.53-1.844.72l-1.131.361-1.027-.7c-.566-.381-1.361-.805-1.781-.932C39.766 21.902 38.131 21.944 37.167 22.283zM33 64C16.432 64 3 50.569 3 34S16.432 4 33 4s30 13.431 30 30S49.568 64 33 64z" style="fill-rule:evenodd;clip-rule:evenodd;fill:"/></svg><span class=new-window><svg height="8" style="enable-background:new 0 0 1000 1000" viewBox="0 0 1e3 1e3" width="8" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M598 128h298v298h-86V274L392 692l-60-60 418-418H598v-86zM810 810V512h86v298c0 46-40 86-86 86H214c-48 0-86-40-86-86V214c0-46 38-86 86-86h298v86H214v596h596z" style="fill-rule:evenodd;clip-rule:evenodd;fill:"/></svg></span></a></li></ul></div></div></nav><div class="tc-l pv6-l pv4 ph3 ph4-ns"><h1 class="dib-ns dn f1-ns fw2 white-90 mb0 lh-title">Genomics of MRSA evolution and persistence on the human body</h1></div></div></header><main class=pb7 role=main><article class="flex-l flex-wrap justify-between mw8 center ph3"><header class="mt4 w-100"><p class="f6 b helvetica tracked">POSTS</p><h1 class="f3-ns f4 athelas mb1">Genomics of MRSA evolution and persistence on the human body</h1><time class="f6 dib tracked" datetime=2019-09-10T06:45:13-04:00>September 10, 2019</time><div class=author-info><p class=author-name>Tim Read</p></div></header><section class="nested-copy-line-height lh-copy serif f4 nested-links nested-img mid-gray pr4-l w-80-l"><p><a href=https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-018-1336-z>Read TD, Petit III RA, Yin Z, Montgomery T, McNulty MC, David MZ. USA300 Staphylococcus aureus persists on multiple body sites following an infection. BMC Microbiol [Internet]. 2018 Dec 5;18(1):206. Available from: https://doi.org/10.1186/s12866-018-1336-z</a></p><p>Microbiologists know surprisingly little about the dynamics of bacterial life on the human body: how strains are acquired, how they move between body sites and how long it takes for them to die off or be replaced. This is because, until recently, we didn’t have the genetic tools that allow us to say whether two bacteria from the same species shared a last common ancestor 2 months ago or 1000 years ago. Affordable genome sequencing introduced over the past 10 years has changed the game and produced a raft of outstanding insights into evolution of bacteria that are on a single person’s body, or “within-host”1–3.</p><p>At the end of last year I published the results of a productive collaboration with Michael David at U. Penn4 that gave us a glimpse into what happens to a notorious human pathogen (<em>Staphylococcus aureus</em>) after it has caused an acute infection. Michael enrolled 30 patients with an infection caused by methicillin-resistant <em>S. aureus</em> (MRSA) strain USA300. This was the first known MRSA infection that each of the subjects had ever had. Shortly after the initial isolate was taken from the site of infection, swab cultures were taken from three potential “colonization” sites - throat, nose and rectum - and we attempted to isolate MRSA. A year later, the patients returned (most of them had shown no signs of infection recurrence) and the same three sites were swabbed. All the initial infecting and colonizing MRSA were then genome sequenced and analyzed using our <a href=https://staphopia.emory.edu>Staphopia</a> analysis pipeline software5.</p><p>All the USA300 genomes were very similar to each other because they all derive from the same bacterial ancestor at the start of the outbreak of community-acquired MRSA in the late 1980s6. However, strains from the same patient tended to have many fewer genetic differences (single nucleotide polymorphisms: SNPs) between them than did strains from different patients. This cutoff of about 40 SNPs in the core genome alignment allowed us to define “intra-subject lineages” (ISLs) that colonized the patients before causing SSTI and persisted after the infection. We can’t rule out that bacteria in ISLs are being lost from a person and then re-acquired from surfaces and other family members - we know from previous studies that households can be colonized by USA300 clones for many years7 - but we presume that the simplest explanation is that bacteria in ISLs are persisting and spreading among sites on humans.</p><p><img src=/images/2019-USA300-behind-the-paper0.png alt="Figure 2 - intra/inter ISL"></p><p>While this study was an observational pilot not powered to draw strong conclusions, it did provide some fascinating insights that we will be following up on. We found that an USA300 ISL <strong>lingers asymptomatically on patients after infection</strong>, although the numbers of patients with the strain declined at the one year sampling point. We also found that during their stay on patients strains <strong>occasionally lost and gained mobile genetic elements</strong> that changed the antibiotic resistance profile. <strong>Persistence was not associated exclusively with any one of the three tested body sites,</strong> and in many cases a strain appeared at a site on the second sampling that wasn&rsquo;t found in the first sampling, <strong>suggesting movement between the sites</strong>. The latter results challenge the usually accepted model that the nose is the primary place where <em>S. aureus</em> lives on the human body when it is not causing disease.</p><p>This paper (along with other within-host studies with Taj Azarian from University of Central Florida8,9) helped us to convince the NIH to give us an RO1 grant to look at MRSA colonization in much greater genetic detail on many more patients. The project, called SEMAPHORE (Study of MRSA, Antibiotics and Persistence, Having the Outcome of Recurrence), is currently in the enrollment stage. We aim to sequence many thousands of genomes to understand how <em>S. aureus</em> that is part of an ISL endures antibiotics, evolves and dies and whether we can determine the genetic factors that predispose a certain subset of patients to serious recurrent MRSA infections.</p><h4 id=references>References</h4><ol><li><p><a href=http://paperpile.com/b/BGdSfL/lpD0>Zhao S, Lieberman TD, Poyet M, Kauffman KM, Gibbons SM, Groussin M, Xavier RJ, Alm EJ. Adaptive Evolution within Gut Microbiomes of Healthy People. Cell Host Microbe [Internet]. 2019 Apr 23; Available from: http://www.sciencedirect.com/science/article/pii/S1931312819301593</a></p></li><li><p><a href=http://paperpile.com/b/BGdSfL/6zf2>Moradigaravand D, Gouliouris T, Blane B, Naydenova P, Ludden C, Crawley C, Brown NM, Török ME, Parkhill J, Peacock SJ. Within-host evolution of Enterococcus faecium during longitudinal carriage and transition to bloodstream infection in immunocompromised patients. Genome Med [Internet]. 2017 Dec 27;9(1):119. Available from: http://dx.doi.org/10.1186/s13073-017-0507-0 PMID: 29282103</a></p></li><li><p><a href=http://paperpile.com/b/BGdSfL/cADQ>Didelot X, Sarah Walker A, Peto TE, Crook DW, Wilson DJ. Within-host evolution of bacterial pathogens. Nat Rev Microbiol [Internet]. Nature Publishing Group; 2016 Jan 19 [cited 2016 Jan 24]; Available from: http://dx.doi.org/10.1038/nrmicro.2015.13</a></p></li><li><p><a href=http://paperpile.com/b/BGdSfL/zEql>Read TD, Petit RA, Yin Z, Montgomery T, McNulty MC, David MZ. USA300 Staphylococcus aureus persists on multiple body sites following an infection. BMC Microbiol [Internet]. 2018 Dec 5;18(1):206. Available from: https://doi.org/10.1186/s12866-018-1336-z</a></p></li><li><p><a href=http://paperpile.com/b/BGdSfL/EBJz>Petit RA 3rd, Read TD. Staphylococcus aureus viewed from the perspective of 40,000+ genomes. PeerJ [Internet]. 2018 Jul 12;6:e5261. Available from: http://dx.doi.org/10.7717/peerj.5261 PMCID: PMC6046195</a></p></li><li><p><a href=http://paperpile.com/b/BGdSfL/XB67>David MZ, Daum RS. Community-associated methicillin-resistant Staphylococcus aureus: epidemiology and clinical consequences of an emerging epidemic. Clin Microbiol Rev [Internet]. 2010 Jul;23(3):616–687. Available from: http://dx.doi.org/10.1128/CMR.00081-09 PMCID: PMC2901661</a></p></li><li><p><a href=http://paperpile.com/b/BGdSfL/5ad2>Alam MT, Read TD, Petit RA, Boyle-Vavra S, Miller LG, Eells SJ, Daum RS, David MZ. Transmission and Microevolution of USA300 MRSA in U.S. Households: Evidence from Whole-Genome Sequencing. MBio [Internet]. American Society for Microbiology; 2015 Mar 10;6(2):e00054–15. Available from: http://dx.doi.org/10.1128/mBio.00054-15 PMID: 25759497</a></p></li><li><p><a href=http://paperpile.com/b/BGdSfL/f8NQ>Azarian T, Ridgway J, Yin Z, David M. Long-term intrahost evolution of methicillin resistant Staphylococcus aureus among cystic fibrosis patients with respiratory carriage. Front Genet [Internet]. 2019 [cited 2019 Apr 4];10:546. Available from: https://www.frontiersin.org/article/10.3389/fgene.2019.00546</a></p></li><li><p><a href=http://paperpile.com/b/BGdSfL/cYdR>Azarian T, Daum RS, Petty LA, Steinbeck JL, Yin Z, Nolan D, Boyle-Vavra S, Hanage WP, Salemi M, David MZ. Intrahost Evolution of Methicillin-Resistant Staphylococcus aureus USA300 Among Individuals With Reoccurring Skin and Soft-Tissue Infections. J Infect Dis [Internet]. 2016 Sep 15;214(6):895–905. Available from: http://dx.doi.org/10.1093/infdis/jiw242 PMCID: PMC4996148</a></p></li></ol><ul class=pa0></ul><div class=mt6></div></section><aside class="w-30-l mt6-l"></aside></article></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://read-lab-confederation.github.io>&copy; 2022 EMERGENT</a><div><a class=resp-sharing-button__link href="https://twitter.com/intent/tweet/?text=Check out EMERGENT, a bacterial genomics and metagenomics blog from @tdread_emory's group!&url=https://read-lab-confederation.github.io" target=_blank rel=noopener aria-label="Share on Twitter"><div class="resp-sharing-button resp-sharing-button--twitter resp-sharing-button--large"><div aria-hidden=true class="resp-sharing-button__icon resp-sharing-button__icon--solid"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M23.44 4.83c-.8.37-1.5.38-2.22.02.93-.56.98-.96 1.32-2.02-.88.52-1.86.9-2.9 1.1-.82-.88-2-1.43-3.3-1.43-2.5.0-4.55 2.04-4.55 4.54.0.36.03.7.1 1.04-3.77-.2-7.12-2-9.36-4.75-.4.67-.6 1.45-.6 2.3.0 1.56.8 2.95 2 3.77-.74-.03-1.44-.23-2.05-.57v.06c0 2.2 1.56 4.03 3.64 4.44-.67.2-1.37.2-2.06.08.58 1.8 2.26 3.12 4.25 3.16C5.78 18.1 3.37 18.74 1 18.46c2 1.3 4.4 2.04 6.97 2.04 8.35.0 12.92-6.92 12.92-12.93.0-.2.0-.4-.02-.6.9-.63 1.96-1.22 2.56-2.14z"/></svg></div>Share on Twitter</div></a></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script><script src=/js/custom.js></script></body></html>