<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>EMERGENT | Introduction to Bioconda on MacOS / Linux</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.79.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css rel=stylesheet><link rel=stylesheet href=/css/custom.css><meta property="og:title" content="Introduction to Bioconda on MacOS / Linux"><meta property="og:description" content="Introduction to Bioconda on MacOS / Linux By Tim Read, October 2019
What? Over the past few years, conda, an open-source package manager, has seen rapid growth in the number of available programs it distributes (via Anaconda Cloud). A useful feature of conda is it allows programs to be separated into channels. Bioconda is one of these channels specializing in maintaining bioinformatic software.
Why? Installing open source software from source (ie the files containing the programs) is hard and time-consuming and often futile."><meta property="og:type" content="article"><meta property="og:url" content="https://read-lab-confederation.github.io/blog/posts/conda-intro/"><meta property="article:published_time" content="2019-10-23T00:00:00+00:00"><meta property="article:modified_time" content="2019-10-23T00:00:00+00:00"><meta itemprop=name content="Introduction to Bioconda on MacOS / Linux"><meta itemprop=description content="Introduction to Bioconda on MacOS / Linux By Tim Read, October 2019
What? Over the past few years, conda, an open-source package manager, has seen rapid growth in the number of available programs it distributes (via Anaconda Cloud). A useful feature of conda is it allows programs to be separated into channels. Bioconda is one of these channels specializing in maintaining bioinformatic software.
Why? Installing open source software from source (ie the files containing the programs) is hard and time-consuming and often futile."><meta itemprop=datePublished content="2019-10-23T00:00:00+00:00"><meta itemprop=dateModified content="2019-10-23T00:00:00+00:00"><meta itemprop=wordCount content="904"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="Introduction to Bioconda on MacOS / Linux"><meta name=twitter:description content="Introduction to Bioconda on MacOS / Linux By Tim Read, October 2019
What? Over the past few years, conda, an open-source package manager, has seen rapid growth in the number of available programs it distributes (via Anaconda Cloud). A useful feature of conda is it allows programs to be separated into channels. Bioconda is one of these channels specializing in maintaining bioinformatic software.
Why? Installing open source software from source (ie the files containing the programs) is hard and time-consuming and often futile."><script type=application/javascript>var doNotTrack=false;if(!doNotTrack){window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;ga('create','UA-148959477-1','auto');ga('send','pageview');}</script><script async src=https://www.google-analytics.com/analytics.js></script><link href=/css/share-button.css rel=stylesheet></head><body class="ma0 avenir bg-near-white"><header class="cover bg-top" style=background-image:url(https://read-lab-confederation.github.io/images/bioconda.png)><div class="pb3-m pb6-l bg-black-60"><nav class="pv3 ph3 ph4-ns tc tl-l" role=navigation><div class="flex-l justify-between items-center center"><a href=https://read-lab-confederation.github.io class="f3 fw2 hover-white no-underline white-90 dib">EMERGENT</a><div class="flex-l items-center"><ul class="pl0 mr3"><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/blog/ title="Blog page">Blog</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/group/ title="Group page">Group</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/research/ title="Research page">Research</a></li><li class="list f5 f4-ns fw4 dib pr1 pr3-ns"></li></ul></div></div></nav><div class="tc-l pv6-l pv4 ph3 ph4-ns"><h1 class="dib-ns dn f1-ns fw2 white-90 mb0 lh-title">Introduction to Bioconda on MacOS / Linux</h1></div></div></header><main class=pb7 role=main><article class="flex-l flex-wrap justify-between mw8 center ph3"><header class="mt4 w-100"><p class="f6 b helvetica tracked">POSTS</p><h1 class="f3-ns f4 athelas mb1">Introduction to Bioconda on MacOS / Linux</h1><time class="f6 dib tracked" datetime=2019-10-23T00:00:00Z>October 23, 2019</time><div class=author-info><p class=author-name>Tim Read</p></div></header><section class="nested-copy-line-height lh-copy serif f4 nested-links nested-img mid-gray pr4-l w-80-l"><h1 id=introduction-to-bioconda-on-macos--linux>Introduction to Bioconda on MacOS / Linux</h1><p>By Tim Read, October 2019</p><h2 id=what>What?</h2><p>Over the past few years, <a href=https://docs.conda.io/en/latest/>conda</a>, an open-source package manager, has seen rapid growth in the number of available programs it distributes (via <a href=https://anaconda.org/>Anaconda Cloud</a>). A useful feature of conda is it allows programs to be separated into channels. <a href=https://bioconda.github.io/>Bioconda</a> is one of these channels specializing in maintaining bioinformatic software.</p><h2 id=why>Why?</h2><p>Installing open source software from source (ie the files containing the programs) is hard and time-consuming and often futile. Bioconda is not perfect but it’s much better than solving the complex riddle of software installation on your own. It’s a one-stop-shop - with more than 6,000 packages (May 2019) developed and ready to deploy. It makes your projects more reproducible and easier to update. Because you don&rsquo;t usually need admin permission to load software onto your own area, it is perfect for use on shared servers.</p><h2 id=installing-miniconda>Installing Miniconda</h2><p>Most Bioconda installs use a stripped-down version of conda called Miniconda. Installation information is <a href=https://conda.io/projects/conda/en/latest/user-guide/install/index.html>here</a>. I performed the following steps on MacOS 10.15 but they should work for other versions (and Linux will also be pretty similar). First download the Python 3 version of Miniconda with the bash (.sh) installer using either <code>wget</code> or <code>curl</code>.</p><pre><code>wget -O ~/miniconda.sh \
    https://repo.anaconda.com/miniconda/Miniconda4-latest-MacOSX-x86_64.sh 
</code></pre><p>Or..</p><pre><code>curl -o ~/miniconda.sh \
    https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
</code></pre><p>Then run the bash installer script.</p><pre><code>bash miniconda.sh 
</code></pre><p>Accept the licenses and follow the instructions. I just accepted all the default install options.</p><p>I opened a new terminal and confirmed install by checking version.</p><pre><code>conda --version

conda 4.7.10
</code></pre><h3 id=for-some-users-installation-in-zshell>For some users: installation in Zshell</h3><p>On mac OSX 10.15 zsh (Zshell) is the new default shell in the terminal application. Here are the <a href=https://support.apple.com/en-us/HT208050>gnarly details</a>. It may also be the default on some Linux systems. You can check which shell you are in by typing,</p><pre><code>echo “$SHELL”
</code></pre><p>If you want to make things easy you can change to the bash shell by this command.</p><pre><code>chsh -s /bin/bash
</code></pre><p>If you use zsh, you will have to set the $PATH environment variable for conda manually. This is done by adding the following line to the end of your zsh profile (typically <code>~/.zshrc</code>).</p><pre><code>export PATH=&quot;/Users/YOUR_USER_NAME/miniconda3/bin:$PATH&quot;
</code></pre><h2 id=using-bioconda>Using Bioconda</h2><p>Bioconda is actually a specific channel within the greater conda project (<a href=https://bioconda.github.io>https://bioconda.github.io</a>). To install the Bioconda channel, I followed the <a href=https://bioconda.github.io/user/install.html#set-up-channels>Bioconda&rsquo;s recommended channel ordering</a> using the commands below in the exact order:</p><pre><code>conda config --add channels defaults
conda config --add channels bioconda
conda config --add channels conda-forge
</code></pre><p>To explore the tools available on Bioconda and get their recipe names, use the search dialog available at the Bioconda site <a href=https://bioconda.github.io/>https://bioconda.github.io/</a></p><h2 id=environments>Environments</h2><p>Environments are a useful feature of conda. These are directories with particular subsets of software. By building environments for a particular purpose you can reduce the confusion created when, for instance, different tools specify different versions of the same software. The default environment is called <em>base</em>. For more information about environments an <a href="https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html?highlight=environment">in-depth discussion is available</a>.</p><h3 id=example-environment-creation>Example Environment Creation</h3><p>For this demonstration, I want to create an environment called <em>Blaster</em> and have contain the programs: <a href="%5Bhttp://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs%5D(http://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs)">NCBI blast</a>, <a href=%5Bhttp://www.drive5.com/muscle/%5D(http://www.drive5.com/muscle/)>muscle aligner</a>, and <a href=%5Bhttps://github.com/kblin/ncbi-genome-download/%5D(https://github.com/kblin/ncbi-genome-download/)>ncbi-genome-download </a>. This can be done with <code>conda create</code>:</p><pre><code>conda create -n Blaster blast muscle ncbi-genome-download
</code></pre><p>The command above will find the packages and install any dependencies into the newly-created <em>Blaster</em> environment.</p><h3 id=activating-and-deactivating-environments>Activating and Deactivating Environments</h3><p>To check the available environments:</p><pre><code>conda info --envs
# conda environments:
#
base                  *  /Users/timothyread/miniconda3
Blaster                  /Users/timothyread/miniconda3/envs/Blaster
</code></pre><p>This reports the environments and their paths and notes which environment is active (the asterisk*). At the moment *base* is the active environment. Because the programs just downloaded are available in the *Blaster* environment, you can’t run them from base.</p><p>To activate <em>Blaster</em>, use <code>conda activate</code>:</p><pre><code>conda activate Blaster
</code></pre><p>This switches the conda environment from <em>base</em> to <em>Blaster</em>. This switch is reflected in <code>conda info</code>:</p><pre><code>conda info --envs
# conda environments:
#
base                     /Users/timothyread/miniconda3
Blaster               *  /Users/timothyread/miniconda3/envs/Blaster
</code></pre><p>This shows <em>Blaster</em> is now the active environment. In this environment, BLAST, Muscle and NCBI Genome Download are now available. You can confirm this by typing, for example,</p><pre><code>blastn -help
</code></pre><p>To deactivate the environment.</p><pre><code>conda deactivate
</code></pre><p>Now the environment is switched back to <em>base</em>. BLAST, Muscle and NCBI Genome Download will no longer be available, unless of course they were already in your PATH.</p><h2 id=getting-more-information>Getting more information</h2><p>To see a list of tools available in your current environment, you can use <code>conda list</code>:</p><pre><code>conda list
# packages in environment at /Users/timothyread/miniconda3:
#
# Name                    Version                   Build  Channel
asn1crypto                0.24.0                   py37_0  
bzip2                     1.0.8                h1de35cc_0  
ca-certificates           2019.5.15                     0  
certifi                   2019.6.16                py37_0  
cffi                      1.12.3           py37hb5b8e2f_0  
chardet                   3.0.4                    py37_1  
conda                     4.7.10                   py37_0  
conda-package-handling    1.3.11                   py37_0  
… etc , etc
</code></pre><h2 id=updating-conda-and-tools>Updating Conda and Tools</h2><p>To update the version of conda or a specific tool, use <code>conda update</code>:</p><pre><code>conda update conda
# OR 
conda update TOOL_NAME
</code></pre><h2 id=strategies-for-using-environments>Strategies for using environments</h2><p>It’s best to download all packages in the environment at once (ie at creation and update time). This is a more efficient way to resolve program dependencies than iterative accumulation, which can lead to very long <em>solving environment</em> wait times.</p><p>Keep separate environments for different projects/workflows. Don’t worry about possible redundant installation of the same package in different environments - these days, programs in general take up insignificant hard drive space compared to data.</p><h2 id=further-information>Further information</h2><p>Obviously there is much more to conda and Bioconda than detailed in this brief intro. To find out more, start with the links in this document or poke around with Google. Have fun!</p><ul class=pa0></ul><div class=mt6></div></section><aside class="w-30-l mt6-l"></aside></article></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://read-lab-confederation.github.io>&copy; 2026 EMERGENT</a><div><a class=resp-sharing-button__link href="https://bsky.app/intent/compose?text=Check out EMERGENT, a bacterial genomics and metagenomics blog from Timothy Read's group! https://read-lab-confederation.github.io" target=_blank rel=noopener aria-label="Share on BlueSky"><div class="resp-sharing-button resp-sharing-button--bluesky resp-sharing-button--large"><div aria-hidden=true class="resp-sharing-button__icon resp-sharing-button__icon--solid"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 10.8c-1.087-2.114-4.046-6.053-6.798-7.995C2.566.944 1.561 1.266.902 1.565.139 1.908.0 3.08.0 3.768c0 .69.378 5.65.624 6.479.815 2.736 3.713 3.66 6.383 3.364.136-.02.275-.039.415-.056-.138.022-.276.04-.415.056-3.912.58-7.387 2.005-2.83 7.078 5.013 5.19 6.87-1.113 7.823-4.308.953 3.195 2.05 9.271 7.733 4.308 4.267-4.308 1.172-6.498-2.74-7.078a8.741 8.741.0 01-.415-.056c.14.017.279.036.415.056 2.67.297 5.568-.628 6.383-3.364.246-.828.624-5.79.624-6.478.0-.69-.139-1.861-.902-2.206-.659-.298-1.664-.62-4.3 1.24C16.046 4.748 13.087 8.687 12 10.8z"/></svg></div>Share on BlueSky</div></a></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script><script src=/js/custom.js></script></body></html>