<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>EMERGENT | Interactions between MRSA and hyperglycemic environments - an undergraduate research journey</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.79.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css rel=stylesheet><link rel=stylesheet href=/css/custom.css><meta property="og:title" content="Interactions between MRSA and hyperglycemic environments - an undergraduate research journey"><meta property="og:description" content="Introduction: A Texan’s First Summer in Atlanta This past summer and over the past year, I have had the opportunity to work in the Read lab as an undergraduate researcher. I am a recent graduate of the Emory College of Arts and Sciences where I received a B.S. in Biology and Philosophy. I came to Emory with the hope of participating in undergraduate research, and at the beginning of my junior year I had that hope realized."><meta property="og:type" content="article"><meta property="og:url" content="https://read-lab-confederation.github.io/blog/posts/alexbog-feb2024/"><meta property="article:published_time" content="2024-02-09T00:00:00+00:00"><meta property="article:modified_time" content="2024-02-09T00:00:00+00:00"><meta itemprop=name content="Interactions between MRSA and hyperglycemic environments - an undergraduate research journey"><meta itemprop=description content="Introduction: A Texan’s First Summer in Atlanta This past summer and over the past year, I have had the opportunity to work in the Read lab as an undergraduate researcher. I am a recent graduate of the Emory College of Arts and Sciences where I received a B.S. in Biology and Philosophy. I came to Emory with the hope of participating in undergraduate research, and at the beginning of my junior year I had that hope realized."><meta itemprop=datePublished content="2024-02-09T00:00:00+00:00"><meta itemprop=dateModified content="2024-02-09T00:00:00+00:00"><meta itemprop=wordCount content="4672"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="Interactions between MRSA and hyperglycemic environments - an undergraduate research journey"><meta name=twitter:description content="Introduction: A Texan’s First Summer in Atlanta This past summer and over the past year, I have had the opportunity to work in the Read lab as an undergraduate researcher. I am a recent graduate of the Emory College of Arts and Sciences where I received a B.S. in Biology and Philosophy. I came to Emory with the hope of participating in undergraduate research, and at the beginning of my junior year I had that hope realized."><script type=application/javascript>var doNotTrack=false;if(!doNotTrack){window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;ga('create','UA-148959477-1','auto');ga('send','pageview');}</script><script async src=https://www.google-analytics.com/analytics.js></script><link href=/css/share-button.css rel=stylesheet></head><body class="ma0 avenir bg-near-white"><header class="cover bg-top" style=background-image:url(https://read-lab-confederation.github.io/images/AB_Figure_2_4_graphs.png)><div class="pb3-m pb6-l bg-black-60"><nav class="pv3 ph3 ph4-ns tc tl-l" role=navigation><div class="flex-l justify-between items-center center"><a href=https://read-lab-confederation.github.io class="f3 fw2 hover-white no-underline white-90 dib">EMERGENT</a><div class="flex-l items-center"><ul class="pl0 mr3"><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/blog/ title="Blog page">Blog</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/group/ title="Group page">Group</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/research/ title="Research page">Research</a></li><li class="list f5 f4-ns fw4 dib pr1 pr3-ns"><a href=https://twitter.com/tdread_emory target=_blank class="link-transition twitter link dib z-999 pt0-l mr1" title="Twitter link" rel=noopener aria-label="follow on Twitter——Opens in a new window"><svg height="24" style="enable-background:new 0 0 67 67" viewBox="0 0 67 67" width="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M37.167 22.283c-2.619.953-4.274 3.411-4.086 6.101l.063 1.038-1.048-.127c-3.813-.487-7.145-2.139-9.974-4.915l-1.383-1.377-.356 1.017c-.754 2.267-.272 4.661 1.299 6.271.838.89.649 1.017-.796.487-.503-.169-.943-.296-.985-.233-.146.149.356 2.076.754 2.839.545 1.06 1.655 2.097 2.871 2.712l1.027.487-1.215.021c-1.173.0-1.215.021-1.089.467.419 1.377 2.074 2.839 3.918 3.475l1.299.444-1.131.678c-1.676.976-3.646 1.526-5.616 1.568C19.775 43.256 19 43.341 19 43.405c0 .211 2.557 1.397 4.044 1.864 4.463 1.377 9.765.783 13.746-1.568 2.829-1.673 5.657-5 6.978-8.221.713-1.716 1.425-4.851 1.425-6.354.0-.975.063-1.102 1.236-2.267.692-.678 1.341-1.419 1.467-1.631.21-.403.188-.403-.88-.043-1.781.636-2.033.551-1.152-.402.649-.678 1.425-1.907 1.425-2.267.0-.063-.314.042-.671.233-.377.212-1.215.53-1.844.72l-1.131.361-1.027-.7c-.566-.381-1.361-.805-1.781-.932C39.766 21.902 38.131 21.944 37.167 22.283zM33 64C16.432 64 3 50.569 3 34S16.432 4 33 4s30 13.431 30 30S49.568 64 33 64z" style="fill-rule:evenodd;clip-rule:evenodd;fill:"/></svg><span class=new-window><svg height="8" style="enable-background:new 0 0 1000 1000" viewBox="0 0 1e3 1e3" width="8" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M598 128h298v298h-86V274L392 692l-60-60 418-418H598v-86zM810 810V512h86v298c0 46-40 86-86 86H214c-48 0-86-40-86-86V214c0-46 38-86 86-86h298v86H214v596h596z" style="fill-rule:evenodd;clip-rule:evenodd;fill:"/></svg></span></a></li></ul></div></div></nav><div class="tc-l pv6-l pv4 ph3 ph4-ns"><h1 class="dib-ns dn f1-ns fw2 white-90 mb0 lh-title">Interactions between MRSA and hyperglycemic environments - an undergraduate research journey</h1></div></div></header><main class=pb7 role=main><article class="flex-l flex-wrap justify-between mw8 center ph3"><header class="mt4 w-100"><p class="f6 b helvetica tracked">POSTS</p><h1 class="f3-ns f4 athelas mb1">Interactions between MRSA and hyperglycemic environments - an undergraduate research journey</h1><time class="f6 dib tracked" datetime=2024-02-09T00:00:00Z>February 9, 2024</time><div class=author-info><p class=author-name>Alex Banul</p></div></header><section class="nested-copy-line-height lh-copy serif f4 nested-links nested-img mid-gray pr4-l w-80-l"><h2 id=introduction-a-texans-first-summer-in-atlanta>Introduction: A Texan’s First Summer in Atlanta</h2><p>This past summer and over the past year, I have had the opportunity to work in the Read lab as an undergraduate researcher. I am a recent graduate of the Emory College of Arts and Sciences where I received a B.S. in Biology and Philosophy. I came to Emory with the hope of participating in undergraduate research, and at the beginning of my junior year I had that hope realized. Under the mentorship of graduate student Brooke Talbot and Dr. Timothy Read, I joined an ongoing study into the lineage-dependent metabolic capacities of methicillin-resistant <em>Staphylococcus aureus</em> (MRSA). Through this project, I learned wet lab skills including bacterial growth and enzyme experimental techniques, as well as dry lab skills such as aligning bacterial genetic sequences, generating a phylogenetic tree, and performing data analysis using the programming language R (v4.0.4; R Core Team 2021). Through Emory’s Summer Undergraduate Research Experience (SURE) program, I designed and executed my own project, following my own research questions and experimental design over an 8-week period. I built on the skills I had developed over the past year and challenged myself further. Again, the guidance and support of Brooke Talbot and all the members of the Read lab provided a firm foundation for me to be confident in my research and endeavor to answer questions about MRSA metabolism and virulence. This blog post aims to both present the findings from my summer of research and some follow-up work done in the fall, as well as illustrate my experiences as an undergraduate performing research in the Read lab.</p><p>MRSA is the leading pathogen isolated from bacterial infections among patients with type 2 diabetes, the most common metabolic illness that is ever increasing in prevalence (Lipsky et al. 1987). MRSA thrive in the nutrient-rich environment of a diabetic host plagued with insulin insensitivity and heightened or uncontrolled blood glucose levels, unable to mount a significant immune response (Thurlow et al. 2020).</p><p><em>S. aureus</em> has the ability to cause infection in a wide range of conditions due to its regulative capabilities over both its metabolism and virulence (ability to infect a host). Some genetic content such as antibiotic resistance genes and immune evasion mechanisms is preserved on mobile genetic elements, aiding in rapid and adaptive evolution to any host environment. Thus, understanding the fitness and virulence profiles of various lineages of MRSA is required for a holistic understanding of the bacteria’s overall infection characteristics in hyperglycemic environments.</p><p>Research has found virulence in hyperglycemic environments to differ among MRSA based on lineage (Sharma‐Kuinkel et al. 2019). Investigations into <em>S. aureus</em> infections in skin abscesses with hyperglycemic conditions found that increased glucose availability significantly enhanced <em>S. aureus</em> virulence potential (Thurlow et al. 2020). This is due to the glucose-sensitive quorum sensing gene regulator <em>S. aureus</em> possesses named <em>Agr</em> (accessory gene regulator), which activates transcription of genes that encode for toxins alongside other virulence factors. Further, two recently acquired glucose transporters in the membrane of <em>S. aureus</em> cause greater uptake of glucose in hyperglycemic environments, and thus quicker activation and maintenance of the <em>Agr</em> system (Thurlow et al. 2020). In contrast, another study found that virulence decreased among <em>S. aureus</em> isolates from diabetic foot ulcers in cases of long-term colonization of diabetic patients (Lavigne et al. 2021). Strains isolated from long-term infections of diabetic foot ulcers were notably lacking major toxinogenic factor-encoding genes; genes that were present in early isolates of the infection. This discrepancy presents a gap in the current understanding of <em>S. aureus</em> gene expression and its connection to blood glucose levels.</p><p>The aim of this research was to understand the role host conditions might play in the evolution of fitness and virulence, and how the glucose environment can affect different lineages of MRSA. This was accomplished primarily through phenotypic investigation. To examine fitness, growth rates of MRSA strains were obtained and compared based on attributes such as lineage and host diabetic status. Virulence was evaluated based on enzymatic activity of superoxide dismutase (SOD), a known virulence factor common to MRSA. Through these analyses, we found that lineage has a significant effect on the growth rate of <em>S. aureus</em>, while host source alone had no observable effect on growth rate. Instead, a conjunctive effect of both lineage and host source was observed. SOD activity was undetectable in our experimental strains, potentially caused by an interaction between pigmentation and the assay’s chemical components. While my summer research concluded there, these results generated many further routes for inquiry, one of which I conducted this past fall involving acetate, a metabolic biproduct of <em>S. aureus</em>.</p><h3 id=meet-the-cast-a-diverse-experimental-cohort>Meet the Cast: A Diverse Experimental Cohort</h3><p>At the time I joined the Read Lab, another ongoing project was examining clonal spread of MRSA bloodstream infections of hospitalized adults. This project utilized a cohort of 105 <em>S. aureus</em> strains that were isolated from patients admitted to either one of two hospitals in the University of Pennsylvania hospital system , and then whole-genome shotgun sequenced using Illumina Miseq and Illumina Next era library kits (Talbot et al., 2022). This collection of strains would both serve as my experimental cohort and my introduction to working with genetic data, as I learned how to translate the strain’s genetic content into a phylogenetic tree (<strong>Figure 1</strong>).</p><p><img src=/images/AB_Figure_1_tree.PNG alt=Fig1></p><p><strong>Figure 1:</strong> Phylogeny of 105 <em>S. aureus</em> strains from University of Pennsylvania hopsital cohort. Colored dots indicate lineage, Blue bars indicate diabetes status, and the red/yellow bars indicate type of acquisition.</p><p>Sequence reads were assembled using shovill (v1.0.1). Sequences were then aligned to the reference genome of strain N315 (GCF_000009645.1) using parsnp (v1.2). Then, the alignment was used to build a maximum likelihood tree using a generalized time reversible model, sampling the alignments with 1000 bootstraps. This tree was then visualized in R using ggtree (v3.5.3) and plotted using ggplot2 (v3.3.6), with metadata visually assigned to it using gheatmap, a function of ggtree. The tree both provided a necessary context within which to evaluate my phenotypic results, as well as allow me to build up the basics of moving around my “dry lab,” using both the Linux command line and bringing its work into RStudio.</p><p>While the patients were admitted to the hospital for reasons unrelated to diabetes (bacteremia), 30% of the selected bloodstream infections were isolated from diabetic hosts. This was one of the characteristics considered when selecting the 24-sample subset for experimentation, ensuring that there was ample representation from both diabetic and non-diabetic hosts. In addition, we chose strains from different lineages (or clonal complexes, as they’re described for <em>S. aureus</em>), which allowed us to evaluate how the two variables—host source and lineage—can either independently or conjunctively contribute to the phenotypic changes observed in the following experimentation.</p><h3 id=approach-1-fitness---the-growth-curves>Approach 1: Fitness - The Growth Curves</h3><p>A variety of techniques exist for the evaluation of bacterial fitness, including biofilm growth, enumeration of colony forming units (CFU), and measuring growth rate via growth curves, to name a few. Prior research in the Read lab has utilized growth curves in detecting fitness differences in MRSA under selection from environmental differences (Su et al., 2021), and as such my mentor found the method to be apt for answering these questions concerning hyperglycemic effects.</p><p>A plate reader generates a growth curve for a liquid culture of bacteria by performing periodic spectrophotometric measurements. By measuring the change in optical density (OD) within a well of bacteria over time, one can estimate the overall change in population density within the well, and from that then determine the growth rate over time. Thus, by growing strains in varying glucose concentrations, a relationship could be observed between fitness and environment and evaluated for variation due to host source or lineage.</p><p>As such, a Chemically-Defined Medium with Glucose (CDMG) was prepared, containing amino acids and salts necessary for bacterial growth and two separate medias with differing concentrations of glucose as the sole carbon source (Hussain et al., 1991; Halsey et al., 2017). 0.9 g/L represented a condition of normal blood glucose concentration, while 2.5 g/L represented hyperglycemic conditions (Fernández-Grajera et al., 2022). The preparation of this media demanded careful attention to detail, as it had to be made fresh every week of experimentation. Ashley Alexander, another member of the lab, aided greatly by letting us borrow components for our own preparation, and providing media for us to test and compare with. While some components could be reused for months after creation, other components such as the amino acids and glucose had to be made “fresh” each week, and the final constitution of the week’s media was typically a multi-day process. Due to the limited nature of the summer, I remember feeling quite anxious each week for the media’s proper creation and ability to promote growth. A week’s delay would have greatly limited my ability to complete what I had planned for the summer. Thankfully, difficulties were fortunately minimal, and after a week or two of media tests with lab strains, all growth curve experiments were conducted successfully without any apparent error due to the media.</p><p>Experimental strains and a lab-stock JE2 strain were taken from frozen stock and plated on TSB agar plates. These plates were then incubated for 24 hours at 37 °C. Then, colonies were selected to inoculate overnight liquid cultures, again using TSB media and incubated for 15 hours at 37 °C with shaking at 225 rpm. A 10-fold dilution was then performed on each culture, and these 10-fold dilutions were incubated and shook for another 3 hours.</p><p>After incubation, the OD value of each 10-fold dilution was obtained, with reads taken at 600 nm. Using this information, 1.5 mL of each glucose condition for each strain was prepared in a 24-well plate at starting OD values of 0.05. This 24-well plate was then put on an Eon BioTek plate reader and incubated at 37 °C. The plate shook at 286 cpm, and the OD of each well was taken every 15 minutes for 18 hours via the analysis software Gen 5 v2.06. Further analyses were performed using R Statistical Software (v4.0.4; R Core Team 2021), specifically the Growthcurver package (v0.3.1). Due to the variation observed after the initial exponential phase of growth (discussed in next paragraph), only the first four hours was considered for the growth rate calculation. The data was also normalized according to the negative controls’ OD values (wells with media and no bacteria) (<strong>Figure 2</strong>).</p><p><img src=/images/AB_Figure_2_4_graphs.png alt=Fig2></p><p><strong>Figure 2:</strong> Each plot above shows growth curves for four different samples, each replicated six times. The dark blue line in each graph is the positive control lab strain JE2. a) A CC72 sample exhibiting normal growth conditions, growing exponentially for the first 4-6 hours then hitting carrying capacity and stagnating. b) A CC78 sample in a high glucose environment, starting its exponential growth at different time points over its 6 replicates. c) A CC5 sample which exhibited a &ldquo;drop&rdquo; morphology shared by the other CC5 strains in the low glucose test condition. The cause of this is unknown, and likely does not reflect actual population quantification. d) The same sample from b. in a low glucose environment, once again presenting uninterpretable results.</p><p>In the figure, each graph represents six replicate growth curves of a single clinical strain in either the high glucose (a & b) or low glucose (c & d) conditions, as well as a lab stock strain of JE2, acting as the positive control (the dark blue line). A typical bacterial growth curve has little activity for the first few hours, then the population drastically increases in an exponential phase, then settles at a constant population in a stationary phase. We found this pattern in both glucose conditions, but also observed a variety of other morphologies. In 2b., it appears that each of the replicates entered their exponential phase at different time points. In 2c, this “drop” morphology was unique to strains from CC5 and only exhibited in the low glucose CDMG. And finally, some growth curves like 1d. were seemingly unintelligible, fluctuating wildly. These variations could be due to biofilm formation, which can inhibit the device’s ability to perform its light-dependent reads. The shaking of the incubator largely prevents this though, and any sedimentation observed in the wells was primarily just byproducts of the <em>S. aureus</em> metabolism. The staggered-growth character of 2b. could also suggest that the population has adapted to a secondary carbon source such as acetate at different points, produced as a byproduct of catabolizing the glucose. This new-found hypothesis would serve as the impetus for the analysis of acetate production I would conduct in the fall.</p><p>Statistical analyses consisted of two-tailed t-tests comparing non-diabetic strains to diabetic ones, CC groups to one another, as well as an ANOVA test evaluating all the clonal complexes (<strong>Figure 3</strong>). We found that lineage had a significant effect on the growth rate of <em>S. aureus</em> in both glucose conditions, with CC8 strains having the highest overall growth rates. Host source alone appeared to have no differences in growth rate in either glucose condition. However, a conjunctive effect of both lineage and host source was observed, with diabetic CC8 strains having higher growth rates than diabetic CC5 strains—a relationship not reflected by the CC8 and CC5 strains from non-diabetic hosts. These results suggest that host evolutionary effects may be more prevalent in certain lineages than others.</p><p><img src=/images/AB_Figure_3_rates.PNG alt=Fig3></p><p><strong>Figure 3:</strong> Growth rates calculated via the growthcurver package in RStudio. Graph on the left seperates the two conditions, and compares growth rates between lineages. Graphs on the right compare growth rates between host diabetes status in the low glucose (top) and high glucose (bottom) conditions.</p><h4 id=approach-15-evaluation-of-acetate-levels>Approach 1.5: Evaluation of Acetate Levels</h4><p>The abnormal growth results exhibited by several CC5 strains when grown in the 0.9 g/L glucose condition (Figure 2c) left members of the lab and myself curious as to its cause. One possible reason for exponential growth followed by rapid population depletion would be an inability to uptake secondary carbon sources once an environment has been depleted of all its glucose. One such secondary carbon source is acetate, a biproduct of the catabolism of glucose and generation of ATP (Somerville et al., 2002). Once all the glucose has been used up in the environment, <em>S. aureus</em> can then activate its tricarboxylic acid (TCA) cycle via transcriptional regulation and begin catabolizing the acetate. One study comfirmed both that <em>S. aureus</em> may be found without the ability to catabolize acetate and that those strains exhibited lower growth than strains which could catabolize acetate (Somerville et al., 2003). This suggested that the CC5 strains that exhibited the &ldquo;drop&rdquo; morphology could lack the ability to effectively catabolize acetate.</p><p>We tested this using the EnzyChrom Acetate Assay Kit, which provides enzymes that react with acetate to produce color. This reaction can be measured via spectrophotometry and correlated to acetate concentration. By conducting a time series experiment and evaluating acetate concentration over time for each strain, we could estimate the change in acetate from the peak of the exponential phase—which should correlate closely with when there is no more glucose in the environment for consumption in the 0.9 g/L condition—and the following hours, determining if the samples uptake or use acetate in the environment for the time period of interest.</p><p>We used only a subset of the strains from the clinical cohort, specifically a CC72 strain with a normal growth morphology, and 3 CC5 strains that exhibited the &ldquo;drop&rdquo; morphology. Strains were first plated on TSB agar and then grown up overnight in TSB for 15 hours at 37 °C. Then, at staggered time points, samples were resuspended in both CDMG conditions to achieve periods of 4, 5, and 6 hours of growth for each sample in each condition. Cells were then vortexed and discarded to obtain supernatant for the assay. Supernatant was pipetted into a 96-well plate, and the reaction components were added to each well. A standard curve provided by the manufacturer was performed to orient the obtained results. After initial OD readings were taken, the plate was stored in the dark and incubated at 25 °C for 30 minutes before OD readings were taken again. Both reads were taken at 570nm, and the difference between the initial and final reads, once correlated to the acetate standard curve, served as a measure of acetate concentration.</p><p>From this, we were able to determine the change in acetate concentration over time for each strain (<strong>Figure 4</strong>). Acetate levels kept increasing over time in the 2.5 g/L glucose condition for all samples. This was to be expected, as there would still likely be glucose in the environment for the samples to consume, and as such the samples would still be releasing acetate into the environment. The glucose-deprived environment of 0.9 g/L saw acetate levels decrease over time for all CC5 strains as well as the JE2 control. This suggests uptake of acetate by the cells. For both conditions, our CC72 sample exhibited no change in acetate levels over time. These results show that the strange growth morphology of the CC5 strains in low-glucose environment is likely not due to an inability to catabolize acetate. We also reaffirm that the carbon metabolism of <em>S. aureus</em> varies across clonal complexes, as the CC72 sample seemed to both release and uptake acetate in equilibrium for both conditions. This unique metabolic profile demands further inquiry, both in controlled mediums such as the CDMG worked with in this study as well as real-world environments.</p><p><img src=/images/AB_Figure_4_acetate.PNG alt=Fig4></p><p><strong>Figure 4:</strong> Change in acetate levels per hour for four experimental strains and JE2 as positive control. Left shows low glucose condition, right shows high glucose condition. In the low glucose environment, the negative rate represents usage and uptake of acetate, while in the high glucose environment this is not discernable.</p><h3 id=approach-2-virulence---sod-quantification>Approach 2: Virulence - SOD Quantification</h3><p>Superoxide dismutase (SOD), which detoxifies reactive oxygen species (ROS), is a virulence factor produced by nearly all MRSA strains. Typically, this enzyme serves the role of detoxifying ROS produced by neutrophils in the immune reaction. In the case of infection of diabetic hosts, this may seem counterintuitive at first, as neutrophils in diabetic individuals produce significantly less ROS (Insuela et al. 2020). However, diabetic individuals actually produce more ROS due to breakdown in mitochondrial function during cellular respiration (Liemburg-Apers et al. 2015), thus increasing overall ROSs. One study found <em>S. aureus</em> that lacked the <em>sodA</em> gene had attenuated growth in high glucose conditions compared to the wild-type (Butrico et al. 2022). This suggests SOD plays a role in increasing <em>S. aureus</em> fitness in infections of hyperglycemic hosts. The aim of this experiment was both to understand the role that host source plays in the evolution of <em>sod</em>, and if glucose environment can affect virulence in the same way across different MRSA.
We evaluated this via an enzymatic activity assay. The EnzyChrom Superoxide Dismutase Assay Kit from Bioassay Systems utilized a xanthine oxidase catalyzed reaction to generate superoxide (<strong>Figure 5</strong>). This then either reacted with a dye to produce color, or with SOD, allowing for colorimetric quantification of SOD activity via spectrophotometry (BioAssay Systems, 2021).</p><p><img src=/images/AB_Figure_5_rxn.PNG alt=Fig5></p><p><strong>Figure 5:</strong> The above chemical interactions represent how the EnzyChrom Superoxide Dismutase Assay Kit functions. SOD reacts with the superoxide (O2-), and any remaining superoxide reacts with a dye to form color. Thus, from color intensity measured via spectrophotometry we may infer SOD concentration.</p><p>Strains were first plated and then grown up overnight in TSB for 15 hours at 37 °C. Then, samples were resuspended in both CDMG conditions and cultured for another 3 hours. Cells were then vortexed and lysed to obtain cell supernatant for the assay. Supernatant was pipetted into a 96-well plate, and the reaction components were added to each well. A standard curve was also performed to orient the obtained results, using SOD enzyme. After initial OD readings were taken, the plate was stored in the dark and incubated at 25 °C for one hour before OD readings were taken again. Both reads were taken at 440nm, and the difference between the initial and final reads, once normalized to the negative control, served as a measure of SOD concentration.</p><p>Having been my first time using the assay kit, and with only a few weeks with which to run the assay, I again felt anxious for the successful execution of the protocol. After what seemed to be a successful application of the kit using lab stock JE2 strains, upon running the assay using the experimental cohort, the results were unexpected and difficult to interpret. All of the experimental wells had more intense coloration than the negative control—a well with no SOD enzyme, and as such should have had the most coloration. As these results fell well outside of the standard curve’s bounds, they could not be used to determine SOD production by the experimental cohort.</p><p>With the deadline of the summer program quickly approaching, I had to move forward and begin analyzing my summer’s earlier work. I was very frustrated initially, as I felt that I hadn’t accomplished what I had set out to do at the start of the summer, and I couldn’t explain the reason for the SOD assay’s strange results. My mentor later hypothesized that the increased coloration could have been a result of higher expression of staphoxanthin due to the xanthine chemical in the reaction. Staphoxanthin is a virulence factor also involved in the detoxification of ROS and responsible for the coloration of <em>S. aureus</em> (Clauditz et al., 2006). What I had thought was initially “failed” results was suddenly an avenue of further inquiry. Despite not making it onto my final poster, this part of my project was most fruitful for my development as a researcher, and a firm reminder that there is no such thing as “useless” results.</p><p>From our experimentation with the stock JE2 strains, we were able to observe SOD activity in two conditions: our high glucose 2.5 g/L condition, and TSB (<strong>Figure 6</strong>). We found a significant difference in SOD enzymatic activity between the two conditions, suggesting that carbon sources might affect MRSA virulence capacity. The 2.5 g/L CDMG only has one carbon source, while TSB is an undefined media with multiple potential carbon sources. While not directly related to the research questions I had set out to answer at the beginning of the summer, this MRSA-carbon source interaction provides another potential topic for further investigation.</p><p><img src=/images/AB_Figure_6_SOD.PNG alt=Fig6></p><p><strong>Figure 6:</strong> Measured SOD concentration of lab stock strain JE2 in a high glucose environment and in TSB. While further testing is required to infer any significant results, this shows how glucose environment may impact SOD concentration in <em>S. aureus</em>.</p><h3 id=conclusions--further-avenues-of-research>Conclusions & Further Avenues of Research</h3><p>Two weeks after I had conducted the final experiment in the lab, I presented a poster I’d created for my summer research at the Undergradute Research Symposium, the capstone event of the SURE program. Given the quick turn-around, I was very thankful to my mentor for ample assistance in designing the layout of the poster and using language that conveyed my research in a succinct and precise manner. I also had a practice run of my presentation with other members in the lab, who all offered both praise and helpful suggestions for revisions.</p><p>Though my summer’s project has come to an end, the overall investigation of MRSA behavior in hyperglycemic conditions still has much left to inquire further. Genomic analysis would reveal the clinical isolates’ SOD production abilities and how the glucose environment might be affecting superoxide levels in <em>S. aureus</em> intracellular content. Also, further testing for CC5 fitness differences in low-glucose environments would be valuable to explore the unique growth curve morphology, which we have found to be unrelated to acetate consumption. Finally, we also hypothesize that the experimental evolution of <em>S. aureus</em> in a hyperglycemic environment could yield changes in the genome and adaptation to the high glucose environment, with particular interest for those CC8 strains.</p><p>This spring semester I will be continuing my work with the Read lab, having concluded my undergraduate career in December. My time in the lab has spurred me on to pursue a graduate degree, either in genetics or bioethics. All of the graduate students in the lab work on compelling and interesting projects, and any interactions I have had with them are both positive and supportive. I was welcomed and treated as a colleague, while also patiently guided as a novice in microbiological research. I feel particularly fortunate, and I look forward to my continued work in the Read Lab with the hope that my experience recounted here encourages more undergraduates to pursue their own research projects.</p><h3 id=references-cited>References Cited</h3><p>BioAssay Systems 2021. EnzyChromTM Acetate Assay Kit (EOAC-100).</p><p>BioAssay Systems 2021. EnzyChromTM Superoxide Dismutase Assay Kit (ESOD-100).</p><p>Butrico, C. E., Klopfenstein, N. B., Green, E. R., Johnson, J. R., Peck, S. H., Ibberson, C. B., Serezani, C. H., & Cassat, J. E. (2022). Hyperglycemia increases the severity of Staphylococcus aureus osteomyelitis and influences bacterial genes required for survival in bone [Preprint]. Microbiology. <a href=https://doi.org/10.1101/2022.11.30.518635>https://doi.org/10.1101/2022.11.30.518635</a></p><p>Clauditz, A., Resch, A., Wieland, K.-P., Peschel, A., & Götz, F. (2006). Staphyloxanthin Plays a Role in the Fitness of Staphylococcus aureus and Its Ability To Cope with Oxidative Stress. Infection and Immunity, 74(8), 4950–4953. <a href=https://doi.org/10.1128/IAI.00204-06>https://doi.org/10.1128/IAI.00204-06</a></p><p>Fernández-Grajera, M., Pacha-Olivenza, M. A., Gallardo-Moreno, A. M., González-Martín, M. L., Pérez-Giraldo, C., & Fernández-Calderón, M. C. (2022). Modification of physico-chemical surface properties and growth of Staphylococcus aureus under hyperglycemia and ketoacidosis conditions. Colloids and Surfaces B: Biointerfaces, 209, 112137. <a href=https://doi.org/10.1016/j.colsurfb.2021.112137>https://doi.org/10.1016/j.colsurfb.2021.112137</a></p><p>Halsey, C. R., Lei, S., Wax, J. K., Lehman, M. K., Nuxoll, A. S., Steinke, L., Sadykov, M., Powers, R., & Fey, P. D. (2017). Amino Acid Catabolism in Staphylococcus aureus and the Function of Carbon Catabolite Repression. mBio, 8(1), e01434-16. <a href=https://doi.org/10.1128/mBio.01434-16>https://doi.org/10.1128/mBio.01434-16</a></p><p>Hussain, M., Hastings, J. G. M., & White, P. J. (1991). A chemically defined medium for slime production by coagulase-negative staphylococci. Journal of Medical Microbiology, 34(3), 143–147. <a href=https://doi.org/10.1099/00222615-34-3-143>https://doi.org/10.1099/00222615-34-3-143</a></p><p>Insuela, D., Coutinho, D., Martins, M., Ferrero, M., & Carvalho, V. (2020). Neutrophil Function Impairment Is a Host Susceptibility Factor to Bacterial Infection in Diabetes. In O. Fuchs & S. Shamsadin Athari (Eds.), Cells of the Immune System. IntechOpen. <a href=https://doi.org/10.5772/intechopen.86600>https://doi.org/10.5772/intechopen.86600</a></p><p>Lavigne, J.-P., Hosny, M., Dunyach-Remy, C., Boutet-Dubois, A., Schuldiner, S., Cellier, N., Yahiaoui-Martinez, A., Molle, V., La Scola, B., Marchandin, H., & Sotto, A. (2021). Long-Term Intrahost Evolution of Staphylococcus aureus Among Diabetic Patients With Foot Infections. Frontiers in Microbiology, 12, 741406. <a href=https://doi.org/10.3389/fmicb.2021.741406>https://doi.org/10.3389/fmicb.2021.741406</a></p><p>Liemburg-Apers, D. C., Willems, P. H. G. M., Koopman, W. J. H., & Grefte, S. (2015). Interactions between mitochondrial reactive oxygen species and cellular glucose metabolism. Archives of Toxicology, 89(8), 1209–1226. <a href=https://doi.org/10.1007/s00204-015-1520-y>https://doi.org/10.1007/s00204-015-1520-y</a></p><p>Lipsky, B. A., Pecoraro, R. E., Chen, M. S., & Koepsell, T. D. (1987). Factors Affecting Staphylococcal Colonization Among NIDDM Outpatients. Diabetes Care, 10(4), 483–486. <a href=https://doi.org/10.2337/diacare.10.4.483>https://doi.org/10.2337/diacare.10.4.483</a></p><p>Sharma‐Kuinkel, B. K., Tkaczyk, C., Bonnell, J., Yu, L., Tovchigrechko, A., Tabor, D. E., Park, L. P., Ruffin, F., Esser, M. T., Sellman, B. R., Fowler, V. G., & Ruzin, A. (2019). Associations of pathogen‐specific and host‐specific characteristics with disease outcome in patients with Staphylococcus aureus bacteremic pneumonia. Clinical & Translational Immunology, 8(7). <a href=https://doi.org/10.1002/cti2.1070>https://doi.org/10.1002/cti2.1070</a></p><p>Su, M., Davis, M. H., Peterson, J., Solis-Lemus, C., Satola, S. W., & Read, T. D. (2021). Effect of genetic background on the evolution of Vancomycin-Intermediate Staphylococcus aureus (VISA). PeerJ, 9, e11764. <a href=https://doi.org/10.7717/peerj.11764>https://doi.org/10.7717/peerj.11764</a></p><p>Somerville, Greg A., et al. “Correlation of Acetate Catabolism and Growth Yield in Staphylococcus Aureus : Implications for Host-Pathogen Interactions.” Infection and Immunity, vol. 71, no. 8, Aug. 2003, pp. 4724–32. <a href=https://doi.org/10.1128/IAI.71.8.4724-4732.2003>https://doi.org/10.1128/IAI.71.8.4724-4732.2003</a></p><p>Somerville, Greg A., et al. “Staphylococcus Aureus Aconitase Inactivation Unexpectedly Inhibits Post-Exponential-Phase Growth and Enhances Stationary-Phase Survival.” Infection and Immunity, vol. 70, no. 11, Nov. 2002, pp. 6373–82. <a href=https://doi.org/10.1128/IAI.70.11.6373-6382.2002>https://doi.org/10.1128/IAI.70.11.6373-6382.2002</a></p><p>Talbot, B. M., Jacko, N. F., Petit, R. A., Pegues, D. A., Shumaker, M. J., Read, T. D., & David, M. Z. (2022). Unsuspected Clonal Spread of Methicillin-Resistant Staphylococcus aureus Causing Bloodstream Infections in Hospitalized Adults Detected Using Whole Genome Sequencing. Clinical Infectious Diseases, ciac339. <a href=https://doi.org/10.1093/cid/ciac339>https://doi.org/10.1093/cid/ciac339</a></p><p>Thurlow, L. R., Stephens, A. C., Hurley, K. E., & Richardson, A. R. (2020). Lack of nutritional immunity in diabetic skin infections promotes Staphylococcus aureus virulence. Science Advances, 6(46), eabc5569. <a href=https://doi.org/10.1126/sciadv.abc5569>https://doi.org/10.1126/sciadv.abc5569</a></p><ul class=pa0></ul><div class=mt6></div></section><aside class="w-30-l mt6-l"></aside></article></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://read-lab-confederation.github.io>&copy; 2024 EMERGENT</a><div><a class=resp-sharing-button__link href="https://twitter.com/intent/tweet/?text=Check out EMERGENT, a bacterial genomics and metagenomics blog from @tdread_emory's group!&url=https://read-lab-confederation.github.io" target=_blank rel=noopener aria-label="Share on Twitter"><div class="resp-sharing-button resp-sharing-button--twitter resp-sharing-button--large"><div aria-hidden=true class="resp-sharing-button__icon resp-sharing-button__icon--solid"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M23.44 4.83c-.8.37-1.5.38-2.22.02.93-.56.98-.96 1.32-2.02-.88.52-1.86.9-2.9 1.1-.82-.88-2-1.43-3.3-1.43-2.5.0-4.55 2.04-4.55 4.54.0.36.03.7.1 1.04-3.77-.2-7.12-2-9.36-4.75-.4.67-.6 1.45-.6 2.3.0 1.56.8 2.95 2 3.77-.74-.03-1.44-.23-2.05-.57v.06c0 2.2 1.56 4.03 3.64 4.44-.67.2-1.37.2-2.06.08.58 1.8 2.26 3.12 4.25 3.16C5.78 18.1 3.37 18.74 1 18.46c2 1.3 4.4 2.04 6.97 2.04 8.35.0 12.92-6.92 12.92-12.93.0-.2.0-.4-.02-.6.9-.63 1.96-1.22 2.56-2.14z"/></svg></div>Share on Twitter</div></a></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script><script src=/js/custom.js></script></body></html>