<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>EMERGENT  | Bacterial Strain names and Biosample IDs</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.54.0" />
    
    
      <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
    

    
    
      <link href="/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css" rel="stylesheet">
    

    
      <link rel="stylesheet" href="/css/custom.css">
    

    
      
    

    

    <meta property="og:title" content="Bacterial Strain names and Biosample IDs" />
<meta property="og:description" content="The public databases maintained by NCBI and other members of the International Nucleotide Sequence Database Collaboration (INSDC) are central resources for bacterial bioinformatic analysis. Two key concepts to know about working with these databases are BioProject and BioSample. These were created1 at NCBI as genome data began to gush in, at the beginning of the 2010s following the development of “second generation” sequencing technologies. The structure is mirrored between the INSDC databases, although IDs for the same data can be different." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://read-lab-confederation.github.io/blog/posts/bacterial_strain_names/" />
<meta property="article:published_time" content="2020-03-15T00:00:00&#43;00:00"/>
<meta property="article:modified_time" content="2020-03-15T00:00:00&#43;00:00"/>

<meta itemprop="name" content="Bacterial Strain names and Biosample IDs">
<meta itemprop="description" content="The public databases maintained by NCBI and other members of the International Nucleotide Sequence Database Collaboration (INSDC) are central resources for bacterial bioinformatic analysis. Two key concepts to know about working with these databases are BioProject and BioSample. These were created1 at NCBI as genome data began to gush in, at the beginning of the 2010s following the development of “second generation” sequencing technologies. The structure is mirrored between the INSDC databases, although IDs for the same data can be different.">


<meta itemprop="datePublished" content="2020-03-15T00:00:00&#43;00:00" />
<meta itemprop="dateModified" content="2020-03-15T00:00:00&#43;00:00" />
<meta itemprop="wordCount" content="1290">



<meta itemprop="keywords" content="" />
<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Bacterial Strain names and Biosample IDs"/>
<meta name="twitter:description" content="The public databases maintained by NCBI and other members of the International Nucleotide Sequence Database Collaboration (INSDC) are central resources for bacterial bioinformatic analysis. Two key concepts to know about working with these databases are BioProject and BioSample. These were created1 at NCBI as genome data began to gush in, at the beginning of the 2010s following the development of “second generation” sequencing technologies. The structure is mirrored between the INSDC databases, although IDs for the same data can be different."/>

<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
	ga('create', 'UA-148959477-1', 'auto');
	
	ga('send', 'pageview');
}
</script>
<script async src='https://www.google-analytics.com/analytics.js'></script>

    <link href="/css/share-button.css" rel="stylesheet">
  </head>

  <body class="ma0 avenir bg-near-white">

    
   
  

  
  
  <header class="cover bg-top" style="background-image: url('https://read-lab-confederation.github.io/images/biosample_logo.png');">
    <div class="pb3-m pb6-l bg-black-60">
      <nav class="pv3 ph3 ph4-ns tc tl-l" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="https://read-lab-confederation.github.io" class="f3 fw2 hover-white no-underline white-90 dib">
      EMERGENT
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f7 f4-ns fw4 dib pr1 pr3-ns">
            <a class="hover-white no-underline white-90" href="/about/" title="About page">
              About
            </a>
          </li>
          
          <li class="list f7 f4-ns fw4 dib pr1 pr3-ns">
            <a class="hover-white no-underline white-90" href="/blog/" title="Blog page">
              Blog
            </a>
          </li>
          
          <li class="list f7 f4-ns fw4 dib pr1 pr3-ns">
            <a class="hover-white no-underline white-90" href="/group/" title="Group page">
              Group
            </a>
          </li>
          
          <li class="list f7 f4-ns fw4 dib pr1 pr3-ns">
            <a class="hover-white no-underline white-90" href="/research/" title="Research page">
              Research
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr1 pr3-ns">



<a href="https://twitter.com/tdread_emory" target="_blank" class="link-transition twitter link dib z-999 pt0-l mr1" title="Twitter link" rel="noopener" aria-label="follow on Twitter——Opens in a new window">
  <svg height="24px"  style="enable-background:new 0 0 67 67;" version="1.1" viewBox="0 0 67 67" width="24px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M37.167,22.283c-2.619,0.953-4.274,3.411-4.086,6.101  l0.063,1.038l-1.048-0.127c-3.813-0.487-7.145-2.139-9.974-4.915l-1.383-1.377l-0.356,1.017c-0.754,2.267-0.272,4.661,1.299,6.271  c0.838,0.89,0.649,1.017-0.796,0.487c-0.503-0.169-0.943-0.296-0.985-0.233c-0.146,0.149,0.356,2.076,0.754,2.839  c0.545,1.06,1.655,2.097,2.871,2.712l1.027,0.487l-1.215,0.021c-1.173,0-1.215,0.021-1.089,0.467  c0.419,1.377,2.074,2.839,3.918,3.475l1.299,0.444l-1.131,0.678c-1.676,0.976-3.646,1.526-5.616,1.568  C19.775,43.256,19,43.341,19,43.405c0,0.211,2.557,1.397,4.044,1.864c4.463,1.377,9.765,0.783,13.746-1.568  c2.829-1.673,5.657-5,6.978-8.221c0.713-1.716,1.425-4.851,1.425-6.354c0-0.975,0.063-1.102,1.236-2.267  c0.692-0.678,1.341-1.419,1.467-1.631c0.21-0.403,0.188-0.403-0.88-0.043c-1.781,0.636-2.033,0.551-1.152-0.402  c0.649-0.678,1.425-1.907,1.425-2.267c0-0.063-0.314,0.042-0.671,0.233c-0.377,0.212-1.215,0.53-1.844,0.72l-1.131,0.361l-1.027-0.7  c-0.566-0.381-1.361-0.805-1.781-0.932C39.766,21.902,38.131,21.944,37.167,22.283z M33,64C16.432,64,3,50.569,3,34S16.432,4,33,4  s30,13.431,30,30S49.568,64,33,64z" style="fill-rule:evenodd;clip-rule:evenodd;fill:;"/></svg>

<span class="new-window"><svg  height="8px"  style="enable-background:new 0 0 1000 1000;" version="1.1" viewBox="0 0 1000 1000" width="8px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" >
<path d="M598 128h298v298h-86v-152l-418 418-60-60 418-418h-152v-86zM810 810v-298h86v298c0 46-40 86-86 86h-596c-48 0-86-40-86-86v-596c0-46 38-86 86-86h298v86h-298v596h596z" style="fill-rule:evenodd;clip-rule:evenodd;fill:;"/>
</svg>
</span></a>








</li>
        </ul>
      
    </div>
  </div>
</nav>

      <div class="tc-l pv6-l pv4 ph3 ph4-ns">
        
          <h1 class="dib-ns dn f1-ns fw2 white-90 mb0 lh-title">Bacterial Strain names and Biosample IDs</h1>
          
        
      </div>
    </div>
  </header>



    <main class="pb7" role="main">
      
  
  <article class="flex-l flex-wrap justify-between mw8 center ph3">

    <header class="mt4 w-100">
      <p class="f6 b helvetica tracked">
          
        POSTS
      </p>
      <h1 class="f1-ns f4 athelas mb1">Bacterial Strain names and Biosample IDs</h1>
      <time class="f6 dib tracked" datetime="2020-03-15T00:00:00Z">March 15, 2020</time>
      

<div class="author-info">
        <p class="author-name">
        
            
            
                
            
                
            
                
            
                
            
                
            
                
            
                
            
                 
                    
                        Tim Read
                    
                
            
                
            
        
    </p>
</div>
      

    

    

    

    

    

    

    

    

    


      
      
      
    </header>

    <section class="nested-copy-line-height lh-copy serif f4 nested-links nested-img mid-gray pr4-l w-two-thirds-l">

<p>The public databases maintained by NCBI and other members of the International Nucleotide Sequence Database Collaboration <a href="http://www.insdc.org/">(INSDC)</a> are central resources for bacterial bioinformatic analysis.  Two key concepts to know about working with these databases are <strong>BioProject</strong> and <strong>BioSample</strong>.  These were created<sup><a href="http://dx.doi.org/10.1093/nar/gkr1163" target="_blank">1</a></sup> at NCBI as genome data began to gush in, at the beginning of the 2010s following the development of “second generation” sequencing technologies.  The structure is mirrored between the INSDC databases, although IDs for the same data can be different.</p>

<p><a href="https://www.ncbi.nlm.nih.gov/books/NBK169438/">BioProject</a> was created to be attached to one or more BioSamples and hold metadata about the study. BioProject IDs are often cited in manuscripts that introduce primary data submission.</p>

<p><a href="https://www.ncbi.nlm.nih.gov/books/NBK169436/">BioSample</a> tracks distinct biological specimens.  In our studies, this typically means a bacterial strain, though it could also be a human cell line, plant or other biological entity.  BioSamples should have a strain name and contain metadata about the strain provenance (e.g. date of isolation, place, host species).  They are also linked to the NCBI Taxonomy database and, if DNA sequencing has been performed, there are links to SRA Experiment and Run tables.  Assemblies submitted to NCBI are linked to BioSample as are many NCBI databases.  Theoretically, a BioSample ID could be linked to one, multiple or zero BioProjects.</p>

<p>Often, our group wants to get all public DNA sequencing runs associated with a species (or a BioProject or other identifier).  This can be easily done through NCBI with the <a href="https://www.ncbi.nlm.nih.gov/Traces/study/">SRA Run Selector tool</a>. (Note, here the “SRA_Study” ID appears to map to a single BioProject).</p>

<h2 id="searching-for-strains-in-published-articles">Searching for strains in published articles</h2>

<p>This is great, but suppose you are looking for publications that mention a particular sequenced strain.  This may happen after making an interesting discovery based on a sequence analysis - for example, you detect an unusual toxin gene in that genome.  You want to find out more about that strain and if any subsequent analysis has been published.  In almost all cases, microbiologists like to use strain names instead of BioSample IDs when they refer to genomes. There are no guidelines for strain names, everyone makes up their own system. Often these names are not good for searching the internet because of their ambiguity - for instance “A12” or “London”. The same name may also be typed differently by different groups, e.g “B+24”,”B 24”, “B-24”, further confounding searches.  Sometimes, when a strain is shipped to another lab, it is given a new accession number - ie the same strain often has multiple names (which themselves may not even uniquely identify that strain in internet searches).  If you try internet searches using the strain name you may get no results at all (perhaps you used the wrong version of the name?), or you get too many and you can’t read through every paper.  Also, can you be sure that the “<em>E. coli</em> A12” you find really the strain that matches the sequenced genome?</p>

<p>One solution would be for microbiologists to create unique strain names (but I don’t know how it would be possible to enforce this). Alternatively, maybe a simpler approach would be to use BioSample ID’s instead of (or in addition to) arbitrary strain names. If you register a genome sequence in a public INSDC database, you have to create a BioSample ID anyway. The BioSample ID is a precise way to assign a unique identifier to bacterial strains.  It can even help with the problem of tracing isolates of the same strain that have been shared between laboratories. If you search the BioSample database for common lab strains you will often see multiple BioSample IDs for a given strain name (e.g. look for “<em>Staphylococcus aureus</em> N315”).  This means the same strain (and mutant variants) have been sequenced in different labs.  This is a <em>good thing</em> because it theoretically enables you to discern different lineages of common strains that may have evolved different phenotypic properties<sup><a href="http://dx.doi.org/10.1111/j.1365-2958.2007.05710.x" target="_blank">2</a></sup>.</p>

<p>Unfortunately, BioSample IDs are rarely used in the text of scientific articles. It is rare to see BioSample ID in papers where the original genome sequencing of the strain was not published.  This is a shame because these follow-on studies on sequenced strains add to the overall metadata picture and can be valuable for linking phenotype to genotype.</p>

<p>We confronted the problem of trying to link literature to strains in 2018 when we were searching for metadata on 43,000+ <em>Staphylococcus aureus</em> public genomes for the Staphopia database<sup><a href="http://dx.doi.org/10.7717/peerj.5261" target="_blank">3</a></sup>.  Based on what was reported to NCBI at the time of database submission, only 6,700 strains could be matched to citing articles.  We built our own database of several thousand PDFs of <em>S. aureus</em> papers compiled using the Paperpile citation manager software and individually matched them against every BioProject (Study), BioSample, Run and Experiment ID using the Mac ‘mdfind’ command.   Using this method we found an additional 5,200 strains linked to citations.  This meant that more than 31,000 strains listed in the NCBI database were effectively invisible to literature search despite our best efforts.</p>

<p>If BioSample IDs were used in text more frequently, how could we find them without going to the trouble of downloading thousands of PDFs?  Google Scholar indexes the full text of many scientific articles and it is possible to identify Project and BioSample IDs.  Similarly, <a href="https://europepmc.org/">PubMed Europe</a>) does full text indexing of 5.8 million open access articles. Importantly, neither appears to index the article’s supplemental material - searches only match IDs that are in the main body of the text.  If you are looking for a particular BioSample ID but are returned a negative, you can infer its presence of the paper by searching for linked IDs - e.g <a href="https://www.ncbi.nlm.nih.gov/sra/docs/submitmeta/">BioProject, Run and Experiment</a>), as we did in the Staphopia analysis above. My group is trying to be more sensitive to this issue. In our recent BioRxiv preprint describing the Bactopia pipeline<sup><a href="https://www.biorxiv.org/content/10.1101/2020.02.28.969394v1.abstract" target="_blank">4</a></sup> where we reanalysed &gt; 1,600 <em>Lactobacillus</em> genomes, we made a list of the 300+ BioProject IDs where we drew data from in the main part of the text where they will be indexed by Google Scholar. This was more efficient than listing 1,600 BioSample IDs, even though in some cases there were strains in BioProjects that were not part of our analysis.</p>

<h2 id="conclusions-recommendations">Conclusions/ Recommendations</h2>

<p>Unfortunately, few microbiologists now use BioSample ID but we could start to adopt them as sort of the ORCID of bacterial strain naming.  We should try to use them in article text (quoting strain and BioSample ID together). If many strains are referenced the article, an alternative would be to include a table of information about strains with name, BioSample ID and other metadata. If possible, these data should not be consigned to Excel sheets in the supplementary information, which don&rsquo;t seem to be indexed by most search engines. Most scientists when writing papers don’t consider how other scientists use internet search engines to find their information.  They should, because text search-ability it is one of the huge advantages of open access publishing.  The easier it is to find terms and names through search engines such as Google Scholar and PubMed, the more your article is read and likely, the more citations your work will get.</p>

<h2 id="references">References</h2>

<ol>
<li><p><a href="http://dx.doi.org/10.1093/nar/gkr1163">Barrett T, Clark K, Gevorgyan R, Gorelenkov V, Gribov E, Karsch-Mizrachi I, Kimelman M, Pruitt KD, Resenchuk S, Tatusova T, Yaschenko E, Ostell J. BioProject and BioSample databases at NCBI: facilitating capture and organization of metadata. Nucleic Acids Res. 2012 Jan;40 Database issue:D57–63.</a></p></li>

<li><p><a href="http://dx.doi.org/10.1111/j.1365-2958.2007.05710.x">Hobman JL, Penn CW, Pallen MJ. Laboratory strains of Escherichia coli: model citizens or deceitful delinquents growing old disgracefully? Mol Microbiol. 2007;64(4):881–885.</a></p></li>

<li><p><a href="http://dx.doi.org/10.7717/peerj.5261">Petit RA III, Read TD. Staphylococcus aureus viewed from the perspective of 40,000+ genomes. PeerJ. 2018 Jul 12;6:e5261.</a></p></li>

<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.02.28.969394v1.abstract">Petit RA, Read TD. Bactopia: a flexible pipeline for complete analysis of bacterial genomes. bioRxiv. 2020. p. 2020.02.28.969394.</a></p></li>
</ol>
<ul class="pa0">
  
</ul>
<div class="mt6">
      
      
      </div>
    </section>

    <aside class="w-30-l mt6-l">




</aside>

  </article>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href="https://read-lab-confederation.github.io" >
    &copy; 2020 EMERGENT
  </a>
    <div>
        <a class="resp-sharing-button__link" href="https://twitter.com/intent/tweet/?text=Check out EMERGENT, a bacterial genomics and metagenomics blog from @tdread_emory's group!&amp;url=https://read-lab-confederation.github.io" target="_blank" rel="noopener" aria-label="Share on Twitter">
          <div class="resp-sharing-button resp-sharing-button--twitter resp-sharing-button--large"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solid">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M23.44 4.83c-.8.37-1.5.38-2.22.02.93-.56.98-.96 1.32-2.02-.88.52-1.86.9-2.9 1.1-.82-.88-2-1.43-3.3-1.43-2.5 0-4.55 2.04-4.55 4.54 0 .36.03.7.1 1.04-3.77-.2-7.12-2-9.36-4.75-.4.67-.6 1.45-.6 2.3 0 1.56.8 2.95 2 3.77-.74-.03-1.44-.23-2.05-.57v.06c0 2.2 1.56 4.03 3.64 4.44-.67.2-1.37.2-2.06.08.58 1.8 2.26 3.12 4.25 3.16C5.78 18.1 3.37 18.74 1 18.46c2 1.3 4.4 2.04 6.97 2.04 8.35 0 12.92-6.92 12.92-12.93 0-.2 0-.4-.02-.6.9-.63 1.96-1.22 2.56-2.14z"/></svg>
            </div>Share on Twitter</div>
        </a>
    </div>
  </div>
</footer>

    
      

  <script src="/dist/js/app.3fc0f988d21662902933.js"></script>


      <script src="/js/custom.js"></script>
    
  </body>
</html>
