<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>EMERGENT | How I freed NCBI antibiogram data from XML</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.79.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css rel=stylesheet><link rel=stylesheet href=/css/custom.css><meta property="og:title" content="How I freed NCBI antibiogram data from XML"><meta property="og:description" content="How I freed NCBI antibiogram data from XML TL;DR  How I made a tidy data frame of >24,000 bacterial BioSample IDs with antibiograms and linked genomic sequence data using a customized R script. Code and data available on the github site.  Antibiograms Antibiotic resistant bacteria continue to become more common worldwide. Diagnosis of antibiotic resistance is a frontline defense for fighting back against these dangerous pathogens. Antibiograms are phenotypic tests used routinely in clinical settings to ascertain the profile of antibiotics that will effectively treat the infection."><meta property="og:type" content="article"><meta property="og:url" content="https://read-lab-confederation.github.io/blog/posts/parsing_ncbi_antibiograms/"><meta property="article:published_time" content="2023-05-18T00:00:00+00:00"><meta property="article:modified_time" content="2023-05-18T00:00:00+00:00"><meta itemprop=name content="How I freed NCBI antibiogram data from XML"><meta itemprop=description content="How I freed NCBI antibiogram data from XML TL;DR  How I made a tidy data frame of >24,000 bacterial BioSample IDs with antibiograms and linked genomic sequence data using a customized R script. Code and data available on the github site.  Antibiograms Antibiotic resistant bacteria continue to become more common worldwide. Diagnosis of antibiotic resistance is a frontline defense for fighting back against these dangerous pathogens. Antibiograms are phenotypic tests used routinely in clinical settings to ascertain the profile of antibiotics that will effectively treat the infection."><meta itemprop=datePublished content="2023-05-18T00:00:00+00:00"><meta itemprop=dateModified content="2023-05-18T00:00:00+00:00"><meta itemprop=wordCount content="1116"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="How I freed NCBI antibiogram data from XML"><meta name=twitter:description content="How I freed NCBI antibiogram data from XML TL;DR  How I made a tidy data frame of >24,000 bacterial BioSample IDs with antibiograms and linked genomic sequence data using a customized R script. Code and data available on the github site.  Antibiograms Antibiotic resistant bacteria continue to become more common worldwide. Diagnosis of antibiotic resistance is a frontline defense for fighting back against these dangerous pathogens. Antibiograms are phenotypic tests used routinely in clinical settings to ascertain the profile of antibiotics that will effectively treat the infection."><script type=application/javascript>var doNotTrack=false;if(!doNotTrack){window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;ga('create','UA-148959477-1','auto');ga('send','pageview');}</script><script async src=https://www.google-analytics.com/analytics.js></script><link href=/css/share-button.css rel=stylesheet></head><body class="ma0 avenir bg-near-white"><header class="cover bg-top" style=background-image:url(https://read-lab-confederation.github.io/images/Fig1-202302.png)><div class="pb3-m pb6-l bg-black-60"><nav class="pv3 ph3 ph4-ns tc tl-l" role=navigation><div class="flex-l justify-between items-center center"><a href=https://read-lab-confederation.github.io class="f3 fw2 hover-white no-underline white-90 dib">EMERGENT</a><div class="flex-l items-center"><ul class="pl0 mr3"><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/blog/ title="Blog page">Blog</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/group/ title="Group page">Group</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/research/ title="Research page">Research</a></li><li class="list f5 f4-ns fw4 dib pr1 pr3-ns"></li></ul></div></div></nav><div class="tc-l pv6-l pv4 ph3 ph4-ns"><h1 class="dib-ns dn f1-ns fw2 white-90 mb0 lh-title">How I freed NCBI antibiogram data from XML</h1></div></div></header><main class=pb7 role=main><article class="flex-l flex-wrap justify-between mw8 center ph3"><header class="mt4 w-100"><p class="f6 b helvetica tracked">POSTS</p><h1 class="f3-ns f4 athelas mb1">How I freed NCBI antibiogram data from XML</h1><time class="f6 dib tracked" datetime=2023-05-18T00:00:00Z>May 18, 2023</time><div class=author-info><p class=author-name>Tim Read</p></div></header><section class="nested-copy-line-height lh-copy serif f4 nested-links nested-img mid-gray pr4-l w-80-l"><h1 id=how-i-freed-ncbi-antibiogram-data-from-xml>How I freed NCBI antibiogram data from XML</h1><h2 id=tldr>TL;DR</h2><ul><li>How I made a tidy data frame of >24,000 bacterial BioSample IDs with antibiograms and linked genomic sequence data using a customized R script.</li><li>Code and data available on the <a href=https://github.com/Read-Lab-Confederation/antibiograms/blob/main/README.md>github site</a>.</li></ul><h2 id=antibiograms>Antibiograms</h2><p>Antibiotic resistant bacteria continue to become more common worldwide. Diagnosis of antibiotic resistance is a frontline defense for fighting back against these dangerous pathogens. Antibiograms are phenotypic tests used routinely in clinical settings to ascertain the profile of antibiotics that will effectively treat the infection. More information about the variety of antibiograms and how they are refined and evolved can be found in these posts from the <a href=https://www.health.state.mn.us/diseases/antibioticresistance/abx/antibiograms.pdf>Minnesota Department of Health </a>and the <a href=https://asm.org/Articles/2022/February/Updating-Breakpoints-in-Antimicrobial-Susceptibili>American Society for Microbiology</a>. My group is interested in using antibiogram phenotypes from whole-genome sequenced strains for developing novel genome-based prediction tools (Su et al; Wissel et al).</p><h2 id=how-to-get-your-hands-on-public-antibiogram-data>How to get your hands on public antibiogram data</h2><p>NCBI has been <a href=https://www.ncbi.nlm.nih.gov/biosample/docs/antibiogram/>collecting antibiograms </a>and linking them to BioSample accessions, which can then be linked to a variety of other databases. (I also looked briefly at the ENA (Matatmoros et al) and PATRIC (Antonopoulos et al) databases but only found a few records when using the general search term “antibiogram”. I may come back to investigate these resources in the future but for now I concentrated on the NCBI.) When I searched the Biosample database for the term”antibiogram” I got 483 hits. These have a list of three-letter codes of drugs for the strain presumably has resistance, eg. “AMP-GEN-SSS-SXT”. I suspected that there was more than that, and actually, if you refine the search using “antibiogram[filter]” you get thousands of hits (thanks Emily!) (Oddly, the results from “antibiogram[filter]” don’t include the ones from the search for just “antibiogram”). Further, you can add a filter for strains which also have an SRA accession (potentially genome sequence data).</p><p><img src=/images/Fig1-202302.png alt=Fig1></p><p>The data is already extensive: - 25,046 BioSample accessions. Looking through typical records, most had a table with 10 columns.</p><p><img src=/images/Fig2-202303.png alt=Fig2></p><p>But a few (exclusively Mycobacterium tuberculosis, I think) had antibiogram tables with 6 columns.</p><p><img src=/images/Fig3-202303.png alt=Fig3></p><p>When I tried to download these data I made the unpleasant finding that the standard text file (itself in a freakishly difficult format for parsing) didn’t contain the antibiotic tables. In order to use the information I would have to work with the XML-formatted output file.</p><p><img src=/images/Fig4-202303.png alt=Fig4></p><h2 id=xml-oh-no>XML: Oh no!</h2><p>It was something of a downer to find that the key data was in an XML format. I have always thought of XML as being particularly awkward to parse and have tried as hard as possible to avoid learning the necessary tools. However, there was no alternative for this project.</p><h2 id=r-scripting-solutions>R scripting solutions</h2><p>My goal was to get the XML-formatted download into a neat R dataframe linking the BioSample ID with one drug result on each row. Luckily I found a <a href=https://rstudio-pubs-static.s3.amazonaws.com/499292_d6edbb19b08f456097333fbf9443f9b7.html>nice blog post </a>by Stephen Howe to help me get started. He used the R XML library, which seems to be the most used tool for XML parsing, although there are several others that are probably at least as good.</p><p>First step was to upload the XML data.</p><pre><code>library(XML)
library(tidyverse)
xml2 &lt;- xmlParse(&quot;./biosample_result-4.xml&quot;)
xmltop2 = xmlRoot(xml2)
NumRecords = xmlSize(xmltop2)
</code></pre><p>The <code>xmlRoot </code>function gives the convenience of accessing the root node of the XML document. I spent a bit of time exploring the structure of the XML database to formulate a plan (and tried a couple of preliminary strategies that didn’t eventually pan out). The major problem was that all the tables were appended as “comments” with unnamed rows and columns. Eventually the solution I hit on was a function that made a dataframe using the row and columns for that record alone. The rows were split based on the column length (which could be 10 or 6) and added to the data frame sequentially. It worked on the records I tested. The function could probably be speeded up (looking at the <code>for</code> loop) but … good enough for now.</p><pre><code>populate_rows &lt;- function(k){
  # k is the record ID from the larger table named  xmltop2
  T_vals &lt;- xpathSApply(xmltop2,paste0('//*[',k,']/Description/Comment/Table/Body/Row/Cell'),xmlValue)
  T_cols &lt;- xpathSApply(xmltop2, paste0('//*[',k,']/Description/Comment/Table/Header/Cell'), xmlValue)
  numcols &lt;- length(T_cols)
  numrows &lt;- length(T_vals)/numcols
  df &lt;- data.frame(BioSample = rep(xmlValue(xmltop2[[k]][['Ids']][[1]]),numrows),Organism = rep(xmlValue(xmltop2[[k]][['Description']][['Organism']]),numrows))   
  # add all the columns
  df[T_cols] &lt;- NA
  #loop to add each row (faster structure possible?)
  for (x in 1:numrows) {
  vs = 1+(x-1)*numcols
  ve = 1+(x-1)*numcols+(numcols-1)
  df[x,3:(numcols+2)]=T_vals[vs:ve]
  }
return(df)
}
</code></pre><p>So the plan was to process all the samples using this function. Then I came across the ugly truth that a minority of samples had two tables in their comments and broke the <code>populate_rows</code> function. I could identify them by having two entries under their “comment” field. I decide to filter these out and possibly come back to them at a later time.</p><pre><code>all_comments &lt;- sapply(1:NumRecords, function(x) xmlSize(xmltop2[[x]][['Description']][['Comment']]))
samples_to_process &lt;- which(all_comments == 1)
table(all_comments)
all_comments
    1     2 
24286   760
</code></pre><p>To parse the results I used the <a href=https://purrr.tidyverse.org>map_df</a> function of the amazing purrr library. Super easy and seamlessly handled the problem of merging data frames of different lengths. As an added bonus, it comes with a progress bar so that I could see that the mapping operation wasn’t stalling.</p><pre><code>combined_df &lt;- map_df(samples_to_process,populate_rows,.progress = TRUE)
</code></pre><h2 id=results>Results</h2><p>This took 8 hours to run (!) so there are lots of reasons to go back and make it more efficient. Parallelization could be possible as well. I spot checked the resulting data frame of > 400,000 rows against random BioSample records and everything seemed OK.</p><p>There are a few things to sort out in the future. I dropped adding a SRA ID field because the XML was not consistent with how it represented this across records - it can be cross-linked through a separate search. In some cases, also because of XML record variability, the Organism field was blank - this can also be fixed later. It may also be worth a separate function to sort out the 760 strains with both types of tables.</p><p>I have made the scripts, input XML file and output in TSV format on a public <a href=https://github.com/Read-Lab-Confederation/antibiograms/blob/main/README.md>github site</a>.</p><h2 id=references>References</h2><p>Su M, Satola SW, Read TD. Genome-Based Prediction of Bacterial Antibiotic Resistance. J Clin Microbiol 2019;57.: <a href=https://doi.org/10.1128/JCM.01405-18.%5D(http://paperpile.com/b/5QSxYp/Ttfs)>https://doi.org/10.1128/JCM.01405-18.](http://paperpile.com/b/5QSxYp/Ttfs)</a></p><p>Wissel EF, Talbot BM, Toyosato NAB, Petit RA, Hertzberg V, Dunlop A, et al. hAMRoaster: a tool for comparing performance of AMR gene detection software. bioRxiv 2023:2022.01.13.476279. <a href=https://doi.org/10.1101/2022.01.13.476279.%5D(http://paperpile.com/b/5QSxYp/4cCC)>https://doi.org/10.1101/2022.01.13.476279.](http://paperpile.com/b/5QSxYp/4cCC)</a></p><p>Matamoros S, Hendriksen RS, Pataki BÁ, Pakseresht N, Rossello M, Silvester N, et al. Accelerating surveillance and research of antimicrobial resistance - an online repository for sharing of antimicrobial susceptibility data associated with whole-genome sequences. Microb Genom 2020. <a href=https://doi.org/10.1099/mgen.0.000342.%5D(http://paperpile.com/b/5QSxYp/0hYg)>https://doi.org/10.1099/mgen.0.000342.](http://paperpile.com/b/5QSxYp/0hYg)</a></p><p>Antonopoulos DA, Assaf R, Aziz RK, Brettin T, Bun C, Conrad N, et al. PATRIC as a unique resource for studying antimicrobial resistance. Brief Bioinform 2017. <a href=https://doi.org/10.1093/bib/bbx083.%5D(http://paperpile.com/b/5QSxYp/zHW1)>https://doi.org/10.1093/bib/bbx083.](http://paperpile.com/b/5QSxYp/zHW1)</a></p><ul class=pa0></ul><div class=mt6></div></section><aside class="w-30-l mt6-l"></aside></article></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://read-lab-confederation.github.io>&copy; 2026 EMERGENT</a><div><a class=resp-sharing-button__link href="https://bsky.app/intent/compose?text=Check out EMERGENT, a bacterial genomics and metagenomics blog from Timothy Read's group! https://read-lab-confederation.github.io" target=_blank rel=noopener aria-label="Share on BlueSky"><div class="resp-sharing-button resp-sharing-button--bluesky resp-sharing-button--large"><div aria-hidden=true class="resp-sharing-button__icon resp-sharing-button__icon--solid"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 10.8c-1.087-2.114-4.046-6.053-6.798-7.995C2.566.944 1.561 1.266.902 1.565.139 1.908.0 3.08.0 3.768c0 .69.378 5.65.624 6.479.815 2.736 3.713 3.66 6.383 3.364.136-.02.275-.039.415-.056-.138.022-.276.04-.415.056-3.912.58-7.387 2.005-2.83 7.078 5.013 5.19 6.87-1.113 7.823-4.308.953 3.195 2.05 9.271 7.733 4.308 4.267-4.308 1.172-6.498-2.74-7.078a8.741 8.741.0 01-.415-.056c.14.017.279.036.415.056 2.67.297 5.568-.628 6.383-3.364.246-.828.624-5.79.624-6.478.0-.69-.139-1.861-.902-2.206-.659-.298-1.664-.62-4.3 1.24C16.046 4.748 13.087 8.687 12 10.8z"/></svg></div>Share on BlueSky</div></a></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script><script src=/js/custom.js></script></body></html>