<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>EMERGENT | An introduction to running Bactopia on Amazon Web Services</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.79.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css rel=stylesheet><link rel=stylesheet href=/css/custom.css><meta property="og:title" content="An introduction to running Bactopia on Amazon Web Services"><meta property="og:description" content="Bactopia is a software pipeline for complete analysis of bacterial genomes, developed by Robert Petit. Bactopia is based on the Nextflow bioinformatic workflow software. One of the many virtues of Nextflow is its adaptability to different compute environments, so I thought I would play around a bit with Amazon Web Services (AWS).
AWS Strategies AWS offers the advantages of paying only for what you ask for/use and having potentially vast scalability."><meta property="og:type" content="article"><meta property="og:url" content="https://read-lab-confederation.github.io/blog/posts/aws-bactopia-intro/"><meta property="article:published_time" content="2021-05-15T00:00:00+00:00"><meta property="article:modified_time" content="2021-05-15T00:00:00+00:00"><meta itemprop=name content="An introduction to running Bactopia on Amazon Web Services"><meta itemprop=description content="Bactopia is a software pipeline for complete analysis of bacterial genomes, developed by Robert Petit. Bactopia is based on the Nextflow bioinformatic workflow software. One of the many virtues of Nextflow is its adaptability to different compute environments, so I thought I would play around a bit with Amazon Web Services (AWS).
AWS Strategies AWS offers the advantages of paying only for what you ask for/use and having potentially vast scalability."><meta itemprop=datePublished content="2021-05-15T00:00:00+00:00"><meta itemprop=dateModified content="2021-05-15T00:00:00+00:00"><meta itemprop=wordCount content="1338"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="An introduction to running Bactopia on Amazon Web Services"><meta name=twitter:description content="Bactopia is a software pipeline for complete analysis of bacterial genomes, developed by Robert Petit. Bactopia is based on the Nextflow bioinformatic workflow software. One of the many virtues of Nextflow is its adaptability to different compute environments, so I thought I would play around a bit with Amazon Web Services (AWS).
AWS Strategies AWS offers the advantages of paying only for what you ask for/use and having potentially vast scalability."><script type=application/javascript>var doNotTrack=false;if(!doNotTrack){window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;ga('create','UA-148959477-1','auto');ga('send','pageview');}</script><script async src=https://www.google-analytics.com/analytics.js></script><link href=/css/share-button.css rel=stylesheet></head><body class="ma0 avenir bg-near-white"><header class="cover bg-top" style=background-image:url(https://read-lab-confederation.github.io/images/aws-bactopia-image3.png)><div class="pb3-m pb6-l bg-black-60"><nav class="pv3 ph3 ph4-ns tc tl-l" role=navigation><div class="flex-l justify-between items-center center"><a href=https://read-lab-confederation.github.io class="f3 fw2 hover-white no-underline white-90 dib">EMERGENT</a><div class="flex-l items-center"><ul class="pl0 mr3"><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/blog/ title="Blog page">Blog</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/group/ title="Group page">Group</a></li><li class="list f7 f4-ns fw4 dib pr1 pr3-ns"><a class="hover-white no-underline white-90" href=/research/ title="Research page">Research</a></li><li class="list f5 f4-ns fw4 dib pr1 pr3-ns"><a href=https://twitter.com/tdread_emory target=_blank class="link-transition twitter link dib z-999 pt0-l mr1" title="Twitter link" rel=noopener aria-label="follow on Twitter——Opens in a new window"><svg height="24" style="enable-background:new 0 0 67 67" viewBox="0 0 67 67" width="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M37.167 22.283c-2.619.953-4.274 3.411-4.086 6.101l.063 1.038-1.048-.127c-3.813-.487-7.145-2.139-9.974-4.915l-1.383-1.377-.356 1.017c-.754 2.267-.272 4.661 1.299 6.271.838.89.649 1.017-.796.487-.503-.169-.943-.296-.985-.233-.146.149.356 2.076.754 2.839.545 1.06 1.655 2.097 2.871 2.712l1.027.487-1.215.021c-1.173.0-1.215.021-1.089.467.419 1.377 2.074 2.839 3.918 3.475l1.299.444-1.131.678c-1.676.976-3.646 1.526-5.616 1.568C19.775 43.256 19 43.341 19 43.405c0 .211 2.557 1.397 4.044 1.864 4.463 1.377 9.765.783 13.746-1.568 2.829-1.673 5.657-5 6.978-8.221.713-1.716 1.425-4.851 1.425-6.354.0-.975.063-1.102 1.236-2.267.692-.678 1.341-1.419 1.467-1.631.21-.403.188-.403-.88-.043-1.781.636-2.033.551-1.152-.402.649-.678 1.425-1.907 1.425-2.267.0-.063-.314.042-.671.233-.377.212-1.215.53-1.844.72l-1.131.361-1.027-.7c-.566-.381-1.361-.805-1.781-.932C39.766 21.902 38.131 21.944 37.167 22.283zM33 64C16.432 64 3 50.569 3 34S16.432 4 33 4s30 13.431 30 30S49.568 64 33 64z" style="fill-rule:evenodd;clip-rule:evenodd;fill:"/></svg><span class=new-window><svg height="8" style="enable-background:new 0 0 1000 1000" viewBox="0 0 1e3 1e3" width="8" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M598 128h298v298h-86V274L392 692l-60-60 418-418H598v-86zM810 810V512h86v298c0 46-40 86-86 86H214c-48 0-86-40-86-86V214c0-46 38-86 86-86h298v86H214v596h596z" style="fill-rule:evenodd;clip-rule:evenodd;fill:"/></svg></span></a></li></ul></div></div></nav><div class="tc-l pv6-l pv4 ph3 ph4-ns"><h1 class="dib-ns dn f1-ns fw2 white-90 mb0 lh-title">An introduction to running Bactopia on Amazon Web Services</h1></div></div></header><main class=pb7 role=main><article class="flex-l flex-wrap justify-between mw8 center ph3"><header class="mt4 w-100"><p class="f6 b helvetica tracked">POSTS</p><h1 class="f3-ns f4 athelas mb1">An introduction to running Bactopia on Amazon Web Services</h1><time class="f6 dib tracked" datetime=2021-05-15T00:00:00Z>May 15, 2021</time><div class=author-info><p class=author-name>Tim Read</p></div></header><section class="nested-copy-line-height lh-copy serif f4 nested-links nested-img mid-gray pr4-l w-80-l"><p><a href=https://bactopia.github.io/>Bactopia</a> is a <a href=https://doi.org/10.1128/mSystems.00190-20>software pipeline for complete analysis of bacterial genomes</a>, developed by <a href=https://www.robertpetit.com/>Robert Petit</a>. Bactopia is based on the <a href=https://www.nextflow.io/>Nextflow</a> bioinformatic workflow software. One of the many virtues of Nextflow is its adaptability to different compute environments, so I thought I would play around a bit with <a href=https://aws.amazon.com/>Amazon Web Services</a> (AWS).</p><h2 id=aws-strategies>AWS Strategies</h2><p>AWS offers the advantages of paying only for what you ask for/use and having potentially vast scalability. This means that if you don&rsquo;t want to build and maintain a large-scale bioinformatics server, you still have the ability to run ambitious projects for relatively modest cash outlay (if you do it right). The disadvantages of AWS stem from its complex ecosystem and the fact that there is always a nagging feeling that screwing up will result in a huge bill somewhere down the road.</p><p>There are infinite strategies for setting up AWS that differ in parameters such as cost, time, complexity and convenience. Here, I used two types of AWS storage: <a href=https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html>S3</a> (Simple Storage Service) and <a href="https://aws.amazon.com/ebs/?ebs-whats-new.sort-by=item.additionalFields.postDateTime&ebs-whats-new.sort-order=desc">EBS</a> (Elastic Block Storage). I think of S3 as sort of a huge hard drive in the cloud, while EBS is like a flash drive attached to an EC2 instance (AWS speak for a rentable machine with processing capability). S3 is cheap and very well backed-up (your data won&rsquo;t get lost). It is very adaptable - it can work with many AWS components and can be accessed through URL or even <a href=https://cloud.netapp.com/blog/amazon-s3-as-a-file-system>mounted on your own server.</a> The EBS advantages are that it is convenient (attached to your EC2 instance) and can be used to save your configurations in the form of snapshots and images.</p><h2 id=getting-started---creating-an-s3-bucket-for-my-fastq-inputs>Getting started - creating an S3 bucket for my FASTQ inputs</h2><p>First thing was to log in AWS and create a new S3 bucket (sort of equivalent to an “uber”-directory name) called “tdr-staph-fastqs”. This bucket was set up as private, meaning only myself and people I authorize can access. Then I uploaded some FASTQ files using the <a href=https://aws.amazon.com/cli/>AWS CLI</a> (Command Line Interface), but there are many other ways to interface with S3. My dataset was 5 randomly chosen paired end Illumina runs of <em>S. aureus</em> from <a href=https://doi.org/10.1186/s12866-018-1336-z>this paper</a>. The total size was 1.8 Gbytes. As AWS charges me $0.023 per GByte per month, I could afford to keep it there while I played around with EC2 compute strategies.</p><p><img src=/images/aws-bactopia-image1.png alt="S3 set up"></p><h2 id=running-all-genomes-on-one-machine>Running all genomes on one machine</h2><p>The simplest approach to start with is to rent a single instance with a big enough hard drive to stage all my data and run everything there. It is not necessarily the cheapest or fastest way but, especially, for small projects like this, the cost differential between strategies is a few cents.</p><p>I selected a m5a.4xlarge instance to launch. This is a fairly powerful machine with 16 CPU and 64 GBytes RAM, which is the ratio we have learned from experience works well with Bactopia jobs on <em>S. aureus</em>. For the machine image I chose the default - ECS-optimized Amazon Linux 2 AMI. I configured the instance with 200 GBytes core EBS storage. This cost $0.69 per hour and at rough estimate, the EBS added another $0.014 per hour.</p><p>Following <a href=https://aws.amazon.com/premiumsupport/knowledge-center/ec2-instance-access-s3-bucket/>this protocol</a>, I also created an IAM (Identity and Access Management) policy and profile to allow full S3 access and attached it to the instance. This means I can use the <a href=https://aws.amazon.com/cli/>AWS CLI</a> on my EC2 instance.</p><p>Once I logged onto the machine I followed a <a href=https://gist.github.com/rpetit3/01f27f52273104bbf609ed5acc9b63a9>gist</a> made by Robert for setting up Bactopia, with a few modifications.</p><p>First I downloaded and set up <a href=https://docs.conda.io/projects/conda/en/latest/index.html>Conda</a> and some other useful utilities. I also installed <code>mamba</code>, which is a faster version of conda.</p><pre><code>$ wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
$ bash Miniconda3-latest-Linux-x86_64.sh -b -f
$ ~/miniconda3/bin/conda init bash
$ source ~/.bashrc
$ rm Miniconda3-latest-Linux-x86_64.sh 
$ conda install mamba -n base -c conda-forge

$ sudo yum install -y bzip2 wget git emacs
$ mamba create -n utils -y -c conda-forge -c bioconda procps-ng sed tar wget which coreutils pigz
</code></pre><p>Then created and activated the Bactopia environment using mamba and conda</p><pre><code>$ mamba create -n bactopia -y -c conda-forge -c bioconda bactopia
$ conda activate bactopia
$ bactopia --version
bactopia 1.7.0
</code></pre><p>Then I took the steps to make the species-specific Bactopia datasets, incorporating the curated <em>S. aureus</em> github repo.</p><pre><code>$ mkdir datasets
$ git clone https://github.com/bactopia-datasets/staphylococcus-aureus.git
$ cp -r staphylococcus-aureus/species-specific/ datasets/
$ rm -rf ../bactopia-datasets/ # clean up
$ bactopia datasets --ariba &quot;card&quot; --species &quot;Staphylococcus aureus&quot; --cpus 4 # takes a few min
</code></pre><p>Next I copied the FASTQ data from S3 and made a file of file names (FOFN), using the <em>bactopia prepare</em> sub-command (as detailed in the <a href=https://bactopia.github.io/usage-basic/#the-fofn-format>bactopia tutorial)</a>.</p><pre><code>$ aws s3 cp s3://tdr-staph-fastqs fastqs/ --recursive
$ bactopia prepare fastqs/ &gt; fastqs.txt
$ more fastqs.txt
sample	runtype	r1	r2	extra
David-10_AGGCAGAA-CTCTCTAT_L001	paired-end	/home/ec2-user/fastqs/David-10_AGGCAGAA-CTCTCTAT_L001_R1.fastq.gz	/home/ec2-user/fastqs/David-10_A
GGCAGAA-CTCTCTAT_L001_R2.fastq.gz	
David-11_AGGCAGAA-TATCCTCT_L001	paired-end	/home/ec2-user/fastqs/David-11_AGGCAGAA-TATCCTCT_L001_R1.fastq.gz	/home/ec2-user/fastqs/David-11_A
GGCAGAA-TATCCTCT_L001_R2.fastq.gz	
David-12_AGGCAGAA-AGAGTAGA_L001	paired-end	/home/ec2-user/fastqs/David-12_AGGCAGAA-AGAGTAGA_L001_R1.fastq.gz	/home/ec2-user/fastqs/David-12_A
GGCAGAA-AGAGTAGA_L001_R2.fastq.gz	
David-13_AGGCAGAA-GTAAGGAG_L001	paired-end	/home/ec2-user/fastqs/David-13_AGGCAGAA-GTAAGGAG_L001_R1.fastq.gz	/home/ec2-user/fastqs/David-13_A
GGCAGAA-GTAAGGAG_L001_R2.fastq.gz	
David-1_CGTACTAG-GCGTAAGA_L001	paired-end	/home/ec2-user/fastqs/David-1_CGTACTAG-GCGTAAGA_L001_R1.fastq.gz	/home/ec2-user/fastqs/David-1_CG
TACTAG-GCGTAAGA_L001_R2.fastq.gz	
</code></pre><p>Finally, I set up access to <a href=https://tower.nf/>Nextflow Tower</a> so that I could monitor the workflow in real time. You need to create an account and <a href=https://www.nextflow.io/blog/2021/nextflow-developer-environment.html>create a secret token</a>.</p><pre><code>$ export TOWER_ACCESS_TOKEN=&lt;secret_token&gt;
</code></pre><p>All the steps above took less than an hour. Now, to launch the analysis of all 5 genomes I used the command below.</p><pre><code>$ bactopia --fastqs fastqs.txt \
         --datasets ./datasets/ \
         --species &quot;Staphylococcus aureus&quot; \
         --coverage 100 \
         --genome_size median \
         --cpus 4 \
         --outdir out_bactopia \
         -with-tower
</code></pre><p>Most of it is fairly obvious and the <a href=https://bactopia.github.io/tutorial/>Bactopia tutorial </a>is an outstanding resource to help understand the parameters. I chose <code>-cpus 4</code> to limit the size of one job as a guess that this would increase <a href=https://bactopia.github.io/usage-basic/#-cpus>the efficiency of the run</a>.</p><p>The first time bactopia is run on a machine it builds 12 conda environments for the different part so the pipeline, so that takes a few minutes ….</p><p>The script generates a URL to Nextflow Tower, so you can watch it all happening in real time. (<em>Beware: Nextflow Tower is hypnotizing and wonderful, and a bit of a productivity killer.</em>) The run was all over in an hour and a half.</p><p><img src=/images/aws-bactopia-image2.png alt="Tower Stats"></p><p>Nextflow produces a lot of nice statistics that the Tower visualizes and it is clear to see that the <em>blast_proteins</em> is a bit of a culprit in slowing down the run. This is because of the large number of proteins in the custom <em>S. aureus</em> database used here. In later versions of the software, we may make some tweaks to this process so that it doesn’t slow down the rest of the pipeline too much.</p><p><img src=/images/aws-bactopia-image3.png alt="Process Stats"></p><p>So all the data is now in the output directory, which can be moved to S3, or downloaded directly from EBS using scp or rsync. These directories can be mined for assemblies, blast results etc. I I also used these to run a couple of <a href=https://bactopia.github.io/bactopia-tools/>bactopia-tools</a> to further summarize and query the data.</p><p>The most basic tool is summary:</p><pre><code>$ bactopia tools summary --bactopia out_bactopia
</code></pre><p>This creates a <code>bactopia-tools </code>folder with summaries of data quality, MLST and antimicrobial resistance hits for each strain. For example,</p><pre><code>$ more bactopia-tools/summary/bactopia/amrfinder/amrfinder-gene-summary.txt
sample_name	AMINOGLYCOSIDE	BETA-LACTAM	FOSFOMYCIN	MACROLIDE	STREPTOTHRICIN	TETRACYCLINE
David-10_AGGCAGAA-CTCTCTAT_L001	True	True	True	True	True	True
David-11_AGGCAGAA-TATCCTCT_L001	True	True	True	True	True	True
David-13_AGGCAGAA-GTAAGGAG_L001	True	True	True	True	True	False
David-1_CGTACTAG-GCGTAAGA_L001	True	True	True	True	True	True
</code></pre><p>Another tool is <code>staph-typer</code> a set of (currently) three typing tools specific for <em>S. aureus</em>.</p><pre><code>$ bactopia tools staph-typer --bactopia out_bactopia
</code></pre><p>The results from this run are put in the <code>bactopia-tools</code> folder under a subfolder for the name of the tool. For example, AgrVATE results show all strains are group 1 <em>agr</em> and probably don’t have frameshifts.</p><pre><code>$ more bactopia-tools/staph-typer/staph-typer/agrvate-results.txt 
#filename	agr_group	match_score	canonical_agrD	multiple_agr	frameshifts
David-10_AGGCAGAA-CTCTCTAT_L001	gp1	13	1	s	0
David-11_AGGCAGAA-TATCCTCT_L001	gp1	13	1	s	u
David-12_AGGCAGAA-AGAGTAGA_L001	gp1	7	1	s	u
David-13_AGGCAGAA-GTAAGGAG_L001	gp1	8	1	s	u
David-1_CGTACTAG-GCGTAAGA_L001	gp1	13	1	s	0
</code></pre><p>After that is done, the instance can be closed , or if there is no immediate need to run more bactopia analysis, terminated.</p><p>(Note - I was charged $1.56 for the 2.5 hour session described here)</p><ul class=pa0></ul><div class=mt6></div></section><aside class="w-30-l mt6-l"></aside></article></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://read-lab-confederation.github.io>&copy; 2021 EMERGENT</a><div><a class=resp-sharing-button__link href="https://twitter.com/intent/tweet/?text=Check out EMERGENT, a bacterial genomics and metagenomics blog from @tdread_emory's group!&url=https://read-lab-confederation.github.io" target=_blank rel=noopener aria-label="Share on Twitter"><div class="resp-sharing-button resp-sharing-button--twitter resp-sharing-button--large"><div aria-hidden=true class="resp-sharing-button__icon resp-sharing-button__icon--solid"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M23.44 4.83c-.8.37-1.5.38-2.22.02.93-.56.98-.96 1.32-2.02-.88.52-1.86.9-2.9 1.1-.82-.88-2-1.43-3.3-1.43-2.5.0-4.55 2.04-4.55 4.54.0.36.03.7.1 1.04-3.77-.2-7.12-2-9.36-4.75-.4.67-.6 1.45-.6 2.3.0 1.56.8 2.95 2 3.77-.74-.03-1.44-.23-2.05-.57v.06c0 2.2 1.56 4.03 3.64 4.44-.67.2-1.37.2-2.06.08.58 1.8 2.26 3.12 4.25 3.16C5.78 18.1 3.37 18.74 1 18.46c2 1.3 4.4 2.04 6.97 2.04 8.35.0 12.92-6.92 12.92-12.93.0-.2.0-.4-.02-.6.9-.63 1.96-1.22 2.56-2.14z"/></svg></div>Share on Twitter</div></a></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script><script src=/js/custom.js></script></body></html>